<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Luke Zappia" />


<title>Twitter coverage of the ABACBS 2018 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Conference Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="BioInfoSummer2018.html">BioInfoSummer2018</a>
    </li>
    <li>
      <a href="ABACBS18.html">ABACBS18</a>
    </li>
    <li>
      <a href="COMBINE18.html">COMBINE18</a>
    </li>
    <li>
      <a href="ASHG18.html">ASHG18</a>
    </li>
    <li>
      <a href="gi2018.html">GI2018</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the ABACBS 2018 conference</h1>
<h4 class="author"><em>Luke Zappia</em></h4>
<h4 class="date"><em>2018-12-07 22:25:57</em></h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Hashtag</td>
<td align="left">#ABACBS2018</td>
</tr>
<tr class="even">
<td align="left">Conference</td>
<td align="left">ABACBS 2018 conference</td>
</tr>
<tr class="odd">
<td align="left">Time Zone</td>
<td align="left">Australia/Melbourne</td>
</tr>
<tr class="even">
<td align="left">Days</td>
<td align="left">c(“2018-11-26”, “2018-11-27”, “2018-11-28”, “2018-11-29”, “2018-11-30”)</td>
</tr>
<tr class="odd">
<td align="left">Accent colour</td>
<td align="left">#dd0c15</td>
</tr>
<tr class="even">
<td align="left">Pallete</td>
<td align="left">Set1</td>
</tr>
<tr class="odd">
<td align="left">k-core</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">Topics k</td>
<td align="left">6</td>
</tr>
<tr class="odd">
<td align="left">Bigram filter</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">Fixed</td>
<td align="left">TRUE</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the ABACBS 2018 conference conference. A total of 1802 tweets from 312 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/ABACBS18/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2018-11-26 - 2018-11-30 in the Australia/Melbourne timezone.</p>
<p><img src="figures/ABACBS18/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/ABACBS18/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/ABACBS18/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/ABACBS18/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/ABACBS18/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/ABACBS18/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/ABACBS18/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/ABACBS18/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/ABACBS18/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/ABACBS18/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/ABACBS18/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/ABACBS18/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
We’ll soon be advertising three new roles, so if you’re a postdoc with a computational PhD (or will be soon), or a bioinformatics or statistics Honours graduate, please get in touch! Tell your postdocs, students, friends! More info: <a href="https://t.co/sRoEMCI5Lv" class="uri">https://t.co/sRoEMCI5Lv</a> #ABACBS2018/#ABACBS18
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>If you are interested in joining my lab for a post-doc in research bioinformatics have a chat to me or message me at #abacbs2018 #COMBINE18</p>
<a href="https://t.co/404MibRFnZ" class="uri">https://t.co/404MibRFnZ</a>
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
<p>#ABACBS2018/#ABACBS18 has been a delightful re-entry for me into the Aus bioinf/comp bio community after 7 years in UK.</p>
I’m back to lead a new research group at <span class="citation">@SVIResearch</span> and <span class="citation">@unimelb</span>, working on all things #singlecell, with a big interest in genetics and methods development.
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Interested in learning more about #singlecell #RNAseq analysis of <span class="citation">@10xgenomics</span> data in #bioconductor? Andy Chen has you covered - course notes at <a href="https://t.co/5tV9lVIqFH" class="uri">https://t.co/5tV9lVIqFH</a> #BioCAsia #abacbs2018
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
Nancy Zhang has given by far the most persuasive argument for an expression recovery (“imputation”) tool for scRNA-seq data I’ve seen. SAVER superiority confirmed by indep’t analysis from <span class="citation">@talandrews</span> and <span class="citation">@m_hemberg</span> in recent F1000 paper #abacbs2018
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Yingxin Lin showing, yet again, that students give amongst the best talks. Here, presenting scMerge for integrating scRNA-seq datasets #abacbs2018 📝: <a href="https://t.co/7Y8fQGqL0T" class="uri">https://t.co/7Y8fQGqL0T</a> 💻: <a href="https://t.co/6lStKA8RWq" class="uri">https://t.co/6lStKA8RWq</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Charity Law is unwell, so won’t be presenting at #abacbs2018, but you can read about her work on exploring intron signal at <a href="https://t.co/ip7TMoN34f" class="uri">https://t.co/ip7TMoN34f</a>. On the upside, the morning session is now back on time :) Get well soon Charity! <a href="https://t.co/lBrXREjZ4e" class="uri">https://t.co/lBrXREjZ4e</a>
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#ABACBS2018 Award Winners - Early-career researcher <span class="citation">@BelindaPhipson</span> - Open science <span class="citation">@torstenseemann</span> - Professional bioinformatics Alex Garnham - Mid-career researcher David Lynn - Senior research fellow Mark Ragan
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Great to see the Illumina developing open source software #ABACBS2018 #byebyeblackbox <a href="https://t.co/GuD4iLys9i" class="uri">https://t.co/GuD4iLys9i</a>
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
BelindaPhipson
</td>
<td style="text-align:left;">
Just wanted to thank <span class="citation">@WEHI_research</span> and <span class="citation">@unimelb</span> for subsidising childcare at #abacbs2018. Fantastic to have this support at conferences.
</td>
<td style="text-align:right;">
12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/ABACBS18/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/ABACBS18/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
<p>#ABACBS2018/#ABACBS18 has been a delightful re-entry for me into the Aus bioinf/comp bio community after 7 years in UK.</p>
I’m back to lead a new research group at <span class="citation">@SVIResearch</span> and <span class="citation">@unimelb</span>, working on all things #singlecell, with a big interest in genetics and methods development.
</td>
<td style="text-align:right;">
87
</td>
</tr>
<tr>
<td style="text-align:left;">
BelindaPhipson
</td>
<td style="text-align:left;">
Just wanted to thank <span class="citation">@WEHI_research</span> and <span class="citation">@unimelb</span> for subsidising childcare at #abacbs2018. Fantastic to have this support at conferences.
</td>
<td style="text-align:right;">
70
</td>
</tr>
<tr>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
Nancy Zhang has given by far the most persuasive argument for an expression recovery (“imputation”) tool for scRNA-seq data I’ve seen. SAVER superiority confirmed by indep’t analysis from <span class="citation">@talandrews</span> and <span class="citation">@m_hemberg</span> in recent F1000 paper #abacbs2018
</td>
<td style="text-align:right;">
59
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Interested in learning more about #singlecell #RNAseq analysis of <span class="citation">@10xgenomics</span> data in #bioconductor? Andy Chen has you covered - course notes at <a href="https://t.co/5tV9lVIqFH" class="uri">https://t.co/5tV9lVIqFH</a> #BioCAsia #abacbs2018
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Managed to get an Oshlack lab photo of the whole group at #abacbs2018 #loveit <a href="https://t.co/dmun9v1Ha7" class="uri">https://t.co/dmun9v1Ha7</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Congratulations to <span class="citation">@BelindaPhipson</span> for winning the #abacbs2018 early career researcher award. There couldn’t be a more deserving winner #soproud
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:left;">
BelindaPhipson
</td>
<td style="text-align:left;">
Feel honoured to receive the ECR award at #abacbs2018. Thank you to my amazing mentors <span class="citation">@AliciaOshlack</span> and Gordon Smyth for nominating me.
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Yingxin Lin showing, yet again, that students give amongst the best talks. Here, presenting scMerge for integrating scRNA-seq datasets #abacbs2018 📝: <a href="https://t.co/7Y8fQGqL0T" class="uri">https://t.co/7Y8fQGqL0T</a> 💻: <a href="https://t.co/6lStKA8RWq" class="uri">https://t.co/6lStKA8RWq</a>
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
<p><span class="citation">@theosysbio</span>: “Data resuscitation”</p>
That’s an apt description of what Bioinformaticians do when they they are not engaged in the experimental design, but rather are asked to save the experiment. #abacbs2018
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:left;">
cabbagesofdoom
</td>
<td style="text-align:left;">
Much deserved #abacbs2018 open source award to <span class="citation">@torstenseemann</span> - I’ve annotated multiple organisms with Prokka, and I’m a fan!
</td>
<td style="text-align:right;">
39
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/ABACBS18/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/ABACBS18/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>my current TweetDeck tab is “#combine18 OR #combine2018 OR #abacbs18 OR #abacbs2018”</p>
maybe I should include the SNPs associated too 😂🤦‍♀️ <a href="https://t.co/iBtt6sLApf" class="uri">https://t.co/iBtt6sLApf</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Apparently the conference program has the hashtag at #abacbs2018 … Best to go with that and not #ABACBS18 I think <a href="https://t.co/Wq3VdV2jSn" class="uri">https://t.co/Wq3VdV2jSn</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
You can vote for <span class="citation">@JovMaksimovic</span> for post-doc rep at the #ABACBS2018 AGM soon <a href="https://t.co/6pEzWwJP3k" class="uri">https://t.co/6pEzWwJP3k</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
I’ve been burned by the short-deadline/part-time issue myself. Something we should all be aware of when promoting #WomenInScience #ABACBS18 #abacbs2018 #COMBINE18 <a href="https://t.co/Xt82J3Jp9X" class="uri">https://t.co/Xt82J3Jp9X</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
singscore #bioconductor software available from <a href="https://t.co/vK9vsLOAnH" class="uri">https://t.co/vK9vsLOAnH</a> #abacbs2018 <a href="https://t.co/W2KJePOvrv" class="uri">https://t.co/W2KJePOvrv</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
Alimahmoudi29
</td>
<td style="text-align:left;">
#abacbs2018 <span class="citation">@S_Foroutan</span> Singscore <a href="https://t.co/wx6GMYKt3r" class="uri">https://t.co/wx6GMYKt3r</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Want more #singlecells in Melbourne next July? #abacbs2018 <a href="https://t.co/fMo1nJ5DvL" class="uri">https://t.co/fMo1nJ5DvL</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Those at #abacbs2018 take note :D <a href="https://t.co/fGFHMlZQ57" class="uri">https://t.co/fGFHMlZQ57</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
PeteHaitch
</td>
<td style="text-align:left;">
Lots of <span class="citation">@nanopore</span> tools amongst this catalogue of long-read bioinformatics tools, compiled by <span class="citation">@shaniAmare</span> and <span class="citation">@QGouil</span> #nanoporeconf #ABACBS2018 <a href="https://t.co/wa3gPJhjNF" class="uri">https://t.co/wa3gPJhjNF</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Nice work <span class="citation">@shaniAmare</span> &amp;amp; <span class="citation">@QGouil</span>. Thanks to <span class="citation">@_lazappi_</span> for providing the template &amp;amp; insights from setting up the amazing <a href="https://t.co/psUuIcoU8B" class="uri">https://t.co/psUuIcoU8B</a> resource. Feedback on our first draft most welcome! Sorry that we couldn’t attend #nanoporeconf as we’re all at #abacbs2018 #BioCAsia <a href="https://t.co/FGLKz3LWH9" class="uri">https://t.co/FGLKz3LWH9</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/ABACBS18/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
Managed to get an Oshlack lab photo of the whole group at #abacbs2018 #loveit <a href="https://t.co/dmun9v1Ha7" class="uri">https://t.co/dmun9v1Ha7</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
Great to see the Illumina developing open source software #ABACBS2018 #byebyeblackbox <a href="https://t.co/GuD4iLys9i" class="uri">https://t.co/GuD4iLys9i</a>
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Charity Law is unwell, so won’t be presenting at #abacbs2018, but you can read about her work on exploring intron signal at <a href="https://t.co/ip7TMoN34f" class="uri">https://t.co/ip7TMoN34f</a>. On the upside, the morning session is now back on time :) Get well soon Charity! <a href="https://t.co/lBrXREjZ4e" class="uri">https://t.co/lBrXREjZ4e</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 international keynote <span class="citation">@DunhamLab</span> “Drivers of aneuploidy and adaptation in yeast” <a href="https://t.co/46wfHZhqMC" class="uri">https://t.co/46wfHZhqMC</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
As the Professional Bioinformaticians Representative at <span class="citation">@abacbs</span> and a co-organiser of the <span class="citation">@RLadiesMelb</span>, I couldn’t be any prouder of our Prof Bioinfo Award winner 2018… Alex Garnham !! 🤗🎉💃🏼💪👏 Congrats lady! 😎👌 #abacbs2018 <a href="https://t.co/ABt02OY59r" class="uri">https://t.co/ABt02OY59r</a>
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
missMethyl #woohoo #abacbs2018 <span class="citation">@JovMaksimovic</span> <span class="citation">@BelindaPhipson</span> <a href="https://t.co/I9g1pV6Gjg" class="uri">https://t.co/I9g1pV6Gjg</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
Congratulations <span class="citation">@_hollywhitfield</span> for winning the <span class="citation">@combine_au</span> Best Talk prize and for sharing it with us at #abacbs2018. A great talk indeed! 👌 And good luck with the upcoming PhD! <span class="citation">@WEHI_research</span> <a href="https://t.co/KeIkV4JTi4" class="uri">https://t.co/KeIkV4JTi4</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>lazappi</em>
</td>
<td style="text-align:left;">
#abacbs2018 Day 2 international keynote from <span class="citation">@ctsa11</span> <a href="https://t.co/NGe43lPpKQ" class="uri">https://t.co/NGe43lPpKQ</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@DoktrNick</span>⁩ shows us a beautiful visualisation of kidney development as a tree, reconstructed from imaging data. Informed development of a generative growth model for this process #abacbs2018 <a href="https://t.co/ZjKFVPAziw" class="uri">https://t.co/ZjKFVPAziw</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
The first ever Clinical Bioinformatics Symposium in Australia was a success with over 60 attendees, 7 great speakers and a fantastic discussion panel to finish it off! 👌🎉 Thx <span class="citation">@abacbs</span> and MGHA for the support. Let’s do it again in 2019! #abacbs2018 <span class="citation">@PeterMacCC</span> <span class="citation">@TeamVCGS</span> <a href="https://t.co/UIzBq55ZZA" class="uri">https://t.co/UIzBq55ZZA</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/Ds98dWvU0AA8xWT.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/ABACBS18/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/ABACBS18/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/ABACBS18/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 Kim-Anh Do now speaking about PRECISE, Personalised cancer-specific integrated estimation. Two main aims in personalised medicine: build more accurate understanding of individual patients, and introduce adaptive treatments to individuals
</td>
<td style="text-align:right;">
0.9904710
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>Leming Shi explains how lucky we are working in life sciences with a lot of things to discover compared to drug developers who have a rough path ahead until it a real difference is made. #PrecisionMedicine</p>
#abacbs2018 <a href="https://t.co/9JAH8eFHXE" class="uri">https://t.co/9JAH8eFHXE</a>
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
.<span class="citation">@TonyPapenfuss</span> rightly acknowledges outstanding support from sponsors to enable provision of childcare at #ABACBS2018. Really important initiative to make it easier for parents to attend the conference! Let’s push to make this a normal provision for conferences.
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<span class="citation">@davisjmcc</span> <span class="citation">@AliciaOshlack</span> <span class="citation">@talandrews</span> <span class="citation">@m_hemberg</span> I’m intrigued by the correction applied for estimating the gene-gene correlation, am I right to assume it’s not just simply taking Pearson over the ‘recovered’ matrix? #ABACBS2018
</td>
<td style="text-align:right;">
0.9860983
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
<span class="citation">@AliciaOshlack</span> So is the goal to maximise the number of speaking slots, or increase equity between those who receive speaking slots? I would argue the latter enhances the conference and community more. #abacbs2018
</td>
<td style="text-align:right;">
0.9840006
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
rebecca_poulos
</td>
<td style="text-align:left;">
Hearing about PRECISE from Kim-Anh Do at #abacbs2018. Here’s the paper: “Personalized Integrated Network Modeling of the Cancer Proteome Atlas” <a href="https://t.co/lNZEm2s30s" class="uri">https://t.co/lNZEm2s30s</a>
</td>
<td style="text-align:right;">
0.9840006
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>Professor Kim-Anh Do explains the traditional difference between biologists and statisticians when they talk about #data_integration.</p>
#abacbs2018 #abacbs18 <a href="https://t.co/sTtJNODhOa" class="uri">https://t.co/sTtJNODhOa</a>
</td>
<td style="text-align:right;">
0.9826950
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
<p>Conference bingo prizes up for grabs. Get your bingo card here. Good luck #confBingo #abacbs2018 #ABACBS18</p>
<a href="https://t.co/TsHcgYWIiU" class="uri">https://t.co/TsHcgYWIiU</a> <a href="https://t.co/EciRSAmGXA" class="uri">https://t.co/EciRSAmGXA</a>
</td>
<td style="text-align:right;">
0.9811574
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
frostickle
</td>
<td style="text-align:left;">
<p>More K-mers, a comment and then a question, and github repos in <span class="citation">@jamesmhogan</span>’s talk. #ABACBS2018 #confBingo</p>
<a href="https://t.co/kawgvbZgpa" class="uri">https://t.co/kawgvbZgpa</a> <a href="https://t.co/xGJG8F7DYl" class="uri">https://t.co/xGJG8F7DYl</a> <a href="https://t.co/MN4w4j8bYS" class="uri">https://t.co/MN4w4j8bYS</a>
</td>
<td style="text-align:right;">
0.9811574
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@stephaniehicks</span>⁩: no smoothies here - battle-droids / R-series droids used instead to explain #scRNAseq technology #BioCAsia #abacbs2018 <a href="https://t.co/tTR8bV4BqV" class="uri">https://t.co/tTR8bV4BqV</a>
</td>
<td style="text-align:right;">
0.9811574
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Maitreya Dunham on rich genomic datasets and thinking about CNV (causative? drivers?) in human disease (aneuploidy, cancer, complex disease). Yeast as a relevant system owing to CNVs affecting many traits in yeast, can perform experimental evolution in a chemostat #abacbs2018
</td>
<td style="text-align:right;">
0.9922191
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@DunhamLab</span>⁩ improved upon results from DNAcopy to segment data along the genome in collaboration with statisticians ⁦<span class="citation">@daniela_witten</span>⁩ and Lucy Gao. Applied 1D Fused Lasso to help distinguish between ‘steppy’ or linear models and map driver genes #abacbs2018 <a href="https://t.co/UdxjsF4j23" class="uri">https://t.co/UdxjsF4j23</a>
</td>
<td style="text-align:right;">
0.9912567
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>Not all machine learning models are created equally! Different assumptions have different results, e.g. fitted DLDA, Random Forest, and SVM on hypothetical set of 2 features</p>
<p>From Session 1 of our June 2018 local workshop <a href="https://t.co/EUs9HoOT19" class="uri">https://t.co/EUs9HoOT19</a></p>
#biocasia #abacbs2018 <a href="https://t.co/nrYC2yGAm2" class="uri">https://t.co/nrYC2yGAm2</a>
</td>
<td style="text-align:right;">
0.9908807
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
paulfharrison
</td>
<td style="text-align:left;">
<p>.<span class="citation">@dunhamlab</span> looking at driver genes vs larger regions of fitness of genes with 1D fused LASSO at #abacbs2018 (if I’m understanding correctly).</p>
Hearing LASSO these days I wonder if Elastic-Net can be used, here would give intermediates between stepwise and smooth curve fit.
</td>
<td style="text-align:right;">
0.9900226
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
DunhamLab
</td>
<td style="text-align:left;">
I’m up first this morning at #abacbs2018! I’ll talk about “Drivers of aneuploidy and adaptation in yeast” focusing on a few datasets we’ve generated that I’d love to set this crowd loose on for more analysis.
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
_VickiJackson
</td>
<td style="text-align:left;">
#abacbs2018 finished for another year. Great speaking to <span class="citation">@egiannoulatou</span> lab members about challenges in sequencing studies. Unrelated to current work, but also enjoyed talk by <span class="citation">@DoktrNick</span> on imaging of developing kidney (reminded me of my previous work in the world of lungs!)
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
You can read more about Tim Peters work applying John Mandel’s row linear model for inter lab comparisons in Bioinformatics and via the consensus #bioconductor package #BioCAsia #abacbs2018 <a href="https://t.co/Z0CXDxkWRw" class="uri">https://t.co/Z0CXDxkWRw</a>
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
methylnick
</td>
<td style="text-align:left;">
What a great way to close my <span class="citation">@abacbs</span> #abacbs2018 time. The inaugural clinical bioinformatics panel session with <span class="citation">@markjcowley</span> Natalie Thorne. Karin Kassahn and Simon Sadedin. Thanks <span class="citation">@RoxaneLegaie</span> for organising the symposium. A success. <a href="https://t.co/cPXqOqkR7I" class="uri">https://t.co/cPXqOqkR7I</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Harry Fong gives us a tour de force on antibody libraries and their analysis with his and ⁦<span class="citation">@malhamdoosh</span>⁩ abseqR #bioconductor package <a href="https://t.co/JXMagASH8w" class="uri">https://t.co/JXMagASH8w</a> for making sense of these data #BioCAsia #abacbs2018 <a href="https://t.co/uILhZoL9py" class="uri">https://t.co/uILhZoL9py</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 <span class="citation">@DunhamLab</span> describes using DNAcopy to sensitively identify ‘fitness breakpoints’ in ingeniously induced tiled aneuploidy in yeast. Learning so much yeast genetics/genomics right now!
</td>
<td style="text-align:right;">
0.9860983
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Just finished the workshop on Differential binding analysis for ChIP-seq data, taught by Alex Garnham. Somehow reminiscent of the RNAseq EdgeR pipeline. #ABACBS2018 An excellent finish to a productive day of learning more R/bioconductor packages for bioinformatics analysis 🤓. <a href="https://t.co/zADS9CRDOg" class="uri">https://t.co/zADS9CRDOg</a>
</td>
<td style="text-align:right;">
0.9916029
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Yunshun Chen from <span class="citation">@WEHI_research</span> taking us through scRNAseq analysis from <span class="citation">@10xgenomics</span> data - 1) mito gene % cut-off ranges between 5-10%, 2) selecting only highly variable genes allows faster PCA/tSNE analysis, 3) starting point for perplexity ~ 5% of all cells. #ABACBS2018 <a href="https://t.co/bK8sNPxm10" class="uri">https://t.co/bK8sNPxm10</a>
</td>
<td style="text-align:right;">
0.9908807
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
cabbagesofdoom
</td>
<td style="text-align:left;">
<span class="citation">@koadman</span> wrt your repeat % Q: v. low gaps levels but also smaller assemblies for the other snake genomes analysed. Will need to dig deeper to tell missing data from possible 10x repeat over-assembly artefact. Are we repeating repeats? #ABACBS2018 #QandA
</td>
<td style="text-align:right;">
0.9900226
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>Richard Edwards on promise of 10x Chromium’s Diploid Phasing to perform comparable to long-read technologies at the cost of short read ones by #demultiplexing using #SNPs.</p>
#abacbs2018 <a href="https://t.co/ZPinB1tqFB" class="uri">https://t.co/ZPinB1tqFB</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
AliciaOshlack
</td>
<td style="text-align:left;">
There are several reasons why rare disease cases remain undiagnosed: 1) variant of unknown significance (lots of work happening here) 2) variants we still don’t have tools (methods, technology)to detect 3) not genetic My lab is interested in 2) #abacbs2018
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
An end-of-the-day shout out to Dr Leming Shi for - IMO - one if the most passionate talks which still gets me thinking about how imperative it is to be focussed on standardisation of the methods we use in order to facilitate their applications. #abacbs2018 <a href="https://t.co/RYXTwNJlqQ" class="uri">https://t.co/RYXTwNJlqQ</a>
</td>
<td style="text-align:right;">
0.9869535
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
kizza_a
</td>
<td style="text-align:left;">
hey <span class="citation">@CFlensburg</span> didnt get to ask at qanda. do you have to be careful with cnv on RNAseq if the samples were pooled? got caught out by pooled mice calling variants on RNAseq #abacbs2018
</td>
<td style="text-align:right;">
0.9851230
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
methylnick
</td>
<td style="text-align:left;">
Another take on the Moore’s Law plot. Chris Saunders from <span class="citation">@illumina</span> #abacbs2018 Compute and runtime important considerations for tools to get to result. <a href="https://t.co/HwWo0Bzkux" class="uri">https://t.co/HwWo0Bzkux</a>
</td>
<td style="text-align:right;">
0.9826950
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
aljabadi
</td>
<td style="text-align:left;">
<p>The “inevitable tSNE plot” possibly distinguishing the tumour cells from others - from Dr Ann-Marrie Patch.</p>
#abacbs2018 <a href="https://t.co/oQGZcemJnU" class="uri">https://t.co/oQGZcemJnU</a>
</td>
<td style="text-align:right;">
0.9826950
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
frostickle
</td>
<td style="text-align:left;">
Afternoon keynote by Leming Shi at #ABACBS2018 - Quality Control and Standardization of Omics and Bioinformatics for Precision Medicine. <a href="https://t.co/ldOdCbPqMJ" class="uri">https://t.co/ldOdCbPqMJ</a>
</td>
<td style="text-align:right;">
0.9826950
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Belinda Phipson discussing growing kidney organoids in a dish and assessing variability between these (kidney organoid maturity is a big contributor towards the variability). Single cell analysis shows highly variable genes, confirming maturity drives variability #ABACBS2018
</td>
<td style="text-align:right;">
0.9912567
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
MelbGenomics
</td>
<td style="text-align:left;">
<span class="citation">@MelbGenomics</span> is proud to sponsored the first-ever Clinical Bioinformatics Symposium in Australia. This emerging cross-disciplinary field will only grow as genomics comes into routine healthcare. Congrats to organisers #ABACBS2018 <span class="citation">@RoxaneLegaie</span> <span class="citation">@TeamVCGS</span> <span class="citation">@PeterMacCC</span> <a href="https://t.co/b8hEvO92I8" class="uri">https://t.co/b8hEvO92I8</a>
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
rathod2912
</td>
<td style="text-align:left;">
#असली_गीता_सार ज्ञानानंद जी को संत रामपाल जी महाराज ने आध्यात्मिक चर्चा में पराजित किया है #PMOIndia #abacbs2018 #tooday <a href="https://t.co/N6nyte2ZnA" class="uri">https://t.co/N6nyte2ZnA</a>
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
JovMaksimovic
</td>
<td style="text-align:left;">
#random thought for the day: as a scatter-brained and consequently scatter-mouthed person I enjoy attending scientific conferences as they often give me the opportunity to speak to people who can eloquently express themselves #goals #BioCAsia #abacbs2018
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
frostickle
</td>
<td style="text-align:left;">
<p>Bioinformatics career panel time at #COMBINE18 with: <span class="citation">@IBMResearch</span> <span class="citation">@bwgoudey</span> <span class="citation">@PeterMacCC</span> <span class="citation">@RoxaneLegaie</span> <span class="citation">@illumina</span> <span class="citation">@ctsa11</span> <span class="citation">@QIMRBerghofer</span> <span class="citation">@IamAMP</span></p>
<p>Good to see a balance in industry vs public hospital vs academic &amp;amp; also gender!</p>
#ABACBS2018 <a href="https://t.co/pzB2UfR6zw" class="uri">https://t.co/pzB2UfR6zw</a>
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
The first ever Clinical Bioinformatics Symposium in Australia was a success with over 60 attendees, 7 great speakers and a fantastic discussion panel to finish it off! 👌🎉 Thx <span class="citation">@abacbs</span> and MGHA for the support. Let’s do it again in 2019! #abacbs2018 <span class="citation">@PeterMacCC</span> <span class="citation">@TeamVCGS</span> <a href="https://t.co/UIzBq55ZZA" class="uri">https://t.co/UIzBq55ZZA</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
RoxaneLegaie
</td>
<td style="text-align:left;">
** Shaming Tweet Warning ** With over 50 people identifying as Professional Bioinformaticians in Australia, and most of them being from Melbourne… How come it was only 5 of us at the Prof Bioinfo get together tonight?? 🤔 Never mind, more food &amp;amp; drinks for us! 😜 #abacbs2018 <a href="https://t.co/2W6euJ4JNz" class="uri">https://t.co/2W6euJ4JNz</a>
</td>
<td style="text-align:right;">
0.9869535
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#abacbs2018 <span class="citation">@steman_research</span> presenting work on ARMET (algorithm for resolving microenvironment transcriptomes). Passes on hierarchical structure of cell types rather than treating independently to deconvolute bulk transcriptomes
</td>
<td style="text-align:right;">
0.9869535
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@DoktrNick</span>⁩ shows us a beautiful visualisation of kidney development as a tree, reconstructed from imaging data. Informed development of a generative growth model for this process #abacbs2018 <a href="https://t.co/ZjKFVPAziw" class="uri">https://t.co/ZjKFVPAziw</a>
</td>
<td style="text-align:right;">
0.9869535
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
BelindaPhipson
</td>
<td style="text-align:left;">
Gordon Smyth describes very neat variance correction for transcript level counts from Salmon/Kallisto leveraging bootstrapping capabilities of both #BioCAsia #abacbs2018
</td>
<td style="text-align:right;">
0.9860983
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
CFlensburg
</td>
<td style="text-align:left;">
<p>#abacbs18 (aka #abacbs2018) going down this week, conference starting tomorrow, and I’ll try to live report a bit. Program here: <a href="https://t.co/CmbEoA7kJX" class="uri">https://t.co/CmbEoA7kJX</a></p>
<p>Feel free to follow/filter/unfollow/mute/block/report as you feel appropriate.</p>
For other live tweeps: <span class="citation">@TweetDeck</span> is awesome.
</td>
<td style="text-align:right;">
0.9912567
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Stephen J. Bent has a story of four moths and working on ancient DNA samples (characteristic damage would be lots of cytosine deamination, base oxidation of purines, shorter fragments). One of these moth samples is almost 100 years old! C&amp;gt;T &amp;amp; G&amp;gt;A at ends is the hint! #abacbs2018
</td>
<td style="text-align:right;">
0.9904710
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
carsweshau
</td>
<td style="text-align:left;">
Improving mutual information (Mc Mahon et Al. J. Roy Soc. 2018), multivariate information measures, partial information decomposition. Switching from R to Julia gave a tremendous computational speed up… (not surprising) #abacbs2018
</td>
<td style="text-align:right;">
0.9895300
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
⁦<span class="citation">@theosysbio</span>:⁩ mutual information, in particular partial information decomposition works better for network reconstruction. Nice results on neuronal differentiation published in Stumpf PS et al. (2017) <a href="https://t.co/0WxhlEoTsP" class="uri">https://t.co/0WxhlEoTsP</a> - first author is a long lost cousin :) #abacbs2018 <a href="https://t.co/M0qYvOvTr4" class="uri">https://t.co/M0qYvOvTr4</a>
</td>
<td style="text-align:right;">
0.9895300
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>#abacbs2018 For those wanting a head-start (or attending other sessions), all BioC Asia ClassifyR workshop material is on the Github:</p>
<p>Used tomorrow (90mins): <a href="https://t.co/P83l9Q8AeB" class="uri">https://t.co/P83l9Q8AeB</a></p>
Prior workshop with more background on classification methods (3hr): <a href="https://t.co/EUs9HoOT19" class="uri">https://t.co/EUs9HoOT19</a>
</td>
<td style="text-align:right;">
0.9895300
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
minouye271
</td>
<td style="text-align:left;">
<p>If anyone at #abacbs2018 is looking for a postdoc in multiomics or polygenic risk scores in Aus or UK, feel free to DM/email me.</p>
<p>The <span class="citation">@CamBakerSGI</span> is recruiting at both its <span class="citation">@BakerResearchAu</span> &amp;amp; <span class="citation">@Cambridge_Uni</span> nodes. Eg <a href="https://t.co/oJ3a6ytDBK" class="uri">https://t.co/oJ3a6ytDBK</a></p>
More info at <a href="https://t.co/R5AQWPf89N" class="uri">https://t.co/R5AQWPf89N</a>
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
<p>#ABACBS2018/#ABACBS18 has been a delightful re-entry for me into the Aus bioinf/comp bio community after 7 years in UK.</p>
I’m back to lead a new research group at <span class="citation">@SVIResearch</span> and <span class="citation">@unimelb</span>, working on all things #singlecell, with a big interest in genetics and methods development.
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
At the Introduction to Linear Modelling Workshop taught by Paul Harrison at #ABACBS2018 - finally, I sort of understand a bit more about the linear modelling behind EdgeR &amp;amp; DEseq2? #slowlylearningmorebioinformatics🐢 <a href="https://t.co/BKZaIFlMo9" class="uri">https://t.co/BKZaIFlMo9</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
methylnick
</td>
<td style="text-align:left;">
Twas a brilliant talk and controls are fundamental to any interpretation of genomic data. It’s a slow adoption but my colleagues are starting to listen. #abacbs2018 #reproducibility #controls #referenceMaterial <a href="https://t.co/49SqJRvO0v" class="uri">https://t.co/49SqJRvO0v</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
YiwenWang_Eva
</td>
<td style="text-align:left;">
Glad that people are interested in how to select methods to adjust for batch effect, even though not for microbiome data. I got many good questions and suggestions today! #abacbs2018 <span class="citation">@tpq__</span> <span class="citation">@annaquagli</span> <span class="citation">@mritchieau</span> Thanks for the picture from my friends! <span class="citation">@youyupei</span> <span class="citation">@jiaan_yu</span> <a href="https://t.co/WlsNTT6gCx" class="uri">https://t.co/WlsNTT6gCx</a>
</td>
<td style="text-align:right;">
0.9869535
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
Nancy Zhang has given by far the most persuasive argument for an expression recovery (“imputation”) tool for scRNA-seq data I’ve seen. SAVER superiority confirmed by indep’t analysis from <span class="citation">@talandrews</span> and <span class="citation">@m_hemberg</span> in recent F1000 paper #abacbs2018
</td>
<td style="text-align:right;">
0.9900226
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
marialenagr
</td>
<td style="text-align:left;">
Learning about the evolution of single cell RNA-seq in a presentation delivered by international keynote speaker Dr Nancy Zhang. Single cell is currently a hot hot hot topic. #ABACBS2018 <span class="citation">@abacbs</span> <a href="https://t.co/NeKLs4iNVd" class="uri">https://t.co/NeKLs4iNVd</a>
</td>
<td style="text-align:right;">
0.9900226
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
KevinWang009
</td>
<td style="text-align:left;">
Taiyun Kim from <span class="citation">@sydneybioinfo</span> group on similarity metrics for single cell RNA-Seq data at #abacbs2018. A surprising result: Correlation based metrics tend to do better than distances metrics like Euclidean distance!
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
hdashnow
</td>
<td style="text-align:left;">
Yingxin Lin: Unlike bulk RNA-seq, by definition in single cell RNA-seq the cells do not have replications. So we need pseudo-replicates - use cells of the same cell type from different batches #ABACBS2018
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
davisjmcc
</td>
<td style="text-align:left;">
Yingxin Lin gives a very nice introduction to scMerge for (you guessed it) merging different batches/experiments of scRNA-seq data. Check out the pre-print: <a href="https://t.co/s4Fu21XPWp" class="uri">https://t.co/s4Fu21XPWp</a>. Check out the software: <a href="https://t.co/8z8kVThTdn" class="uri">https://t.co/8z8kVThTdn</a> #ABACBS2018
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
My poster at #abacbs2018 is online now, check it out if you didn’t get a chance earlier <a href="https://t.co/2AgKTXssrd" class="uri">https://t.co/2AgKTXssrd</a> ‘Examining cell quality in association with specific cell types for droplet-based single cell RNA-Sequencing’
</td>
<td style="text-align:right;">
0.9889863
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
The very last talk of #abacbs2018 is <span class="citation">@TheEllisPatrick</span> speaking about Differential correlation across ranked samples for single-cell RNA-sequencing data. Software available: <a href="https://t.co/YDf2SMGo2C" class="uri">https://t.co/YDf2SMGo2C</a> DCARS work and TCGA application in Bioinformatics: <a href="https://t.co/MxFgN6BFNX" class="uri">https://t.co/MxFgN6BFNX</a>
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
sydneybioinfo
</td>
<td style="text-align:left;">
Several <span class="citation">@sydneybioinfo</span> members are in attendance at #abacbs2018 with talks + posters. Give these people a follow &amp;amp; come say Hi! <span class="citation">@KevinWang009</span> Taiyun Kim (talk today), Hani Kim, <span class="citation">@LinYingxin</span> <span class="citation">@rylmb1</span> <span class="citation">@shazanfar</span> <span class="citation">@TheEllisPatrick</span> Nick Canete, Heeva Baharlou, Kitty Lo, Dario Strbenac
</td>
<td style="text-align:right;">
0.9883829
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
annaquagli
</td>
<td style="text-align:left;">
Clare Sloggert presenting Reduce to visualise high dimensional data interactively! Written with Plotly Dash: code in #python 🐍and create great web-apps based on #JavaScript #abacbs2018
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
mritchieau
</td>
<td style="text-align:left;">
Yi Jin Liew: coral CpG methylation is very different to patterns seen in mammals. Mostly occurs in gene bodies and much rarer (~ 7%) at CpG islands compared to humans (~ 60-80%) #abacbs2018 <a href="https://t.co/PjWS0noQZa" class="uri">https://t.co/PjWS0noQZa</a>
</td>
<td style="text-align:right;">
0.9877097
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">10</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">ABACBS2018</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/alexyfyf/abacbs2018" class="uri">https://github.com/alexyfyf/abacbs2018</a></td>
</tr>
<tr class="even">
<td align="left">abseqPy</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/malhamdoosh/abseqpy" class="uri">https://github.com/malhamdoosh/abseqpy</a></td>
</tr>
<tr class="odd">
<td align="left">abseqR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/abseqR" class="uri">https://bioconductor.org/packages/abseqR</a></td>
</tr>
<tr class="even">
<td align="left">badvisualisations</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/biovisml/badvisualisations" class="uri">https://bitbucket.org/biovisml/badvisualisations</a></td>
</tr>
<tr class="odd">
<td align="left">bazam</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ssadedin/bazam" class="uri">https://github.com/ssadedin/bazam</a></td>
</tr>
<tr class="even">
<td align="left">bionitio</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/bionitio-team/bionitio/issues/70" class="uri">https://github.com/bionitio-team/bionitio/issues/70</a></td>
</tr>
<tr class="odd">
<td align="left">bionix</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/papenfusslab/bionix" class="uri">https://github.com/papenfusslab/bionix</a></td>
</tr>
<tr class="even">
<td align="left">canvas</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/canvas" class="uri">https://github.com/illumina/canvas</a></td>
</tr>
<tr class="odd">
<td align="left">celaref</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/monashbioinformaticsplatform/celaref" class="uri">https://github.com/monashbioinformaticsplatform/celaref</a></td>
</tr>
<tr class="even">
<td align="left">celaref</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/celaref" class="uri">https://bioconductor.org/packages/celaref</a></td>
</tr>
<tr class="odd">
<td align="left">ClassifyR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/ClassifyR" class="uri">https://bioconductor.org/packages/ClassifyR</a></td>
</tr>
<tr class="even">
<td align="left">clustree</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lazappi/clustree" class="uri">https://github.com/lazappi/clustree</a></td>
</tr>
<tr class="odd">
<td align="left">clustree</td>
<td align="left">CRAN</td>
<td align="left"><a href="https://CRAN.R-project.org/package=clustree" class="uri">https://CRAN.R-project.org/package=clustree</a></td>
</tr>
<tr class="even">
<td align="left">crisscrosslinker</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/egmg726/crisscrosslinker" class="uri">https://github.com/egmg726/crisscrosslinker</a></td>
</tr>
<tr class="odd">
<td align="left">DCARS</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/shazanfar/dcars" class="uri">https://github.com/shazanfar/dcars</a></td>
</tr>
<tr class="even">
<td align="left">dee2</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/markziemann/dee2" class="uri">https://github.com/markziemann/dee2</a></td>
</tr>
<tr class="odd">
<td align="left">deepvariant</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/google/deepvariant" class="uri">https://github.com/google/deepvariant</a></td>
</tr>
<tr class="even">
<td align="left">diffHicTutorialBioCAsia2018</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/hcoughlan/diffhictutorialbiocasia2018" class="uri">https://github.com/hcoughlan/diffhictutorialbiocasia2018</a></td>
</tr>
<tr class="odd">
<td align="left">dv-trio</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vccri/dv-trio" class="uri">https://github.com/vccri/dv-trio</a></td>
</tr>
<tr class="even">
<td align="left">edgeR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/edgeR" class="uri">https://bioconductor.org/packages/edgeR</a></td>
</tr>
<tr class="odd">
<td align="left">EndoVir</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/ncbi-hackathons/endovir" class="uri">https://github.com/ncbi-hackathons/endovir</a></td>
</tr>
<tr class="even">
<td align="left">ExpansionHunter</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/expansionhunter" class="uri">https://github.com/illumina/expansionhunter</a></td>
</tr>
<tr class="odd">
<td align="left">GeneMates</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/wanyuac/genemates" class="uri">https://github.com/wanyuac/genemates</a></td>
</tr>
<tr class="even">
<td align="left">havic</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/schultzm/havic" class="uri">https://github.com/schultzm/havic</a></td>
</tr>
<tr class="odd">
<td align="left">limma</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/limma" class="uri">https://bioconductor.org/packages/limma</a></td>
</tr>
<tr class="even">
<td align="left">localWorkshop</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/localworkshop" class="uri">https://github.com/sydneybiox/localworkshop</a></td>
</tr>
<tr class="odd">
<td align="left">manta</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/manta" class="uri">https://github.com/illumina/manta</a></td>
</tr>
<tr class="even">
<td align="left">Mixture_model_feature_selection</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/marcelacespedes/mixture_model_feature_selection" class="uri">https://github.com/marcelacespedes/mixture_model_feature_selection</a></td>
</tr>
<tr class="odd">
<td align="left">paragraph</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/paragraph" class="uri">https://github.com/illumina/paragraph</a></td>
</tr>
<tr class="even">
<td align="left">parktree</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tchappell/parktree" class="uri">https://github.com/tchappell/parktree</a></td>
</tr>
<tr class="odd">
<td align="left">Pathway-Visualiser</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/algarnham/pathway-visualiser" class="uri">https://github.com/algarnham/pathway-visualiser</a></td>
</tr>
<tr class="even">
<td align="left">plyranges</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sa-lee/plyranges" class="uri">https://github.com/sa-lee/plyranges</a></td>
</tr>
<tr class="odd">
<td align="left">Polaris</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/polaris" class="uri">https://github.com/illumina/polaris</a></td>
</tr>
<tr class="even">
<td align="left">reduct</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/claresloggett/reduct" class="uri">https://github.com/claresloggett/reduct</a></td>
</tr>
<tr class="odd">
<td align="left">SAVER</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/mohuangx/saver" class="uri">https://github.com/mohuangx/saver</a></td>
</tr>
<tr class="even">
<td align="left">Scavenger</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vccri/scavenger" class="uri">https://github.com/vccri/scavenger</a></td>
</tr>
<tr class="odd">
<td align="left">scClustBench</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/taiyunkim/scclustbench" class="uri">https://github.com/taiyunkim/scclustbench</a></td>
</tr>
<tr class="even">
<td align="left">scMerge</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/scmerge/" class="uri">https://github.com/sydneybiox/scmerge/</a></td>
</tr>
<tr class="odd">
<td align="left">scMerge</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/scmerge" class="uri">https://github.com/sydneybiox/scmerge</a></td>
</tr>
<tr class="even">
<td align="left">SigClust</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/tchappell/sigclust" class="uri">https://github.com/tchappell/sigclust</a></td>
</tr>
<tr class="odd">
<td align="left">singscore</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/singscore" class="uri">https://bioconductor.org/packages/singscore</a></td>
</tr>
<tr class="even">
<td align="left">STAR</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/alexdobin/star" class="uri">https://github.com/alexdobin/star</a></td>
</tr>
<tr class="odd">
<td align="left">strelka</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/illumina/strelka" class="uri">https://github.com/illumina/strelka</a></td>
</tr>
<tr class="even">
<td align="left">STRetch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/oshlack/stretch#boomboomtsch" class="uri">https://github.com/oshlack/stretch#boomboomtsch</a></td>
</tr>
<tr class="odd">
<td align="left">superFreq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/christofferflensburg/superfreq" class="uri">https://github.com/christofferflensburg/superfreq</a></td>
</tr>
<tr class="even">
<td align="left">superFreq</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/christofferflensburg/superfreq/blob/master/readme.md" class="uri">https://github.com/christofferflensburg/superfreq/blob/master/readme.md</a></td>
</tr>
<tr class="odd">
<td align="left">TrainClassifyR</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/sydneybiox/trainclassifyr" class="uri">https://github.com/sydneybiox/trainclassifyr</a></td>
</tr>
<tr class="even">
<td align="left">trndiff</td>
<td align="left">BitBucket</td>
<td align="left"><a href="https://bitbucket.org/biovisml/trndiff.2015" class="uri">https://bitbucket.org/biovisml/trndiff.2015</a></td>
</tr>
<tr class="odd">
<td align="left">ularcirc</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vccri/ularcirc" class="uri">https://github.com/vccri/ularcirc</a></td>
</tr>
<tr class="even">
<td align="left">vsearch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/torognes/vsearch" class="uri">https://github.com/torognes/vsearch</a></td>
</tr>
<tr class="odd">
<td align="left">working_with_dna_meth</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/lyijin/working_with_dna_meth" class="uri">https://github.com/lyijin/working_with_dna_meth</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## ─ Session info ──────────────────────────────────────────────────────────
##  setting  value                       
##  version  R version 3.5.1 (2018-07-02)
##  os       macOS High Sierra 10.13.6   
##  system   x86_64, darwin15.6.0        
##  ui       X11                         
##  language (EN)                        
##  collate  en_AU.UTF-8                 
##  ctype    en_AU.UTF-8                 
##  tz       Australia/Perth             
##  date     2018-12-07                  
## 
## ─ Packages ──────────────────────────────────────────────────────────────
##  package      * version  date       lib source        
##  assertthat     0.2.0    2017-04-11 [1] CRAN (R 3.5.0)
##  backports      1.1.2    2017-12-13 [1] CRAN (R 3.5.0)
##  base64enc      0.1-3    2015-07-28 [1] CRAN (R 3.5.0)
##  bindr          0.1.1    2018-03-13 [1] CRAN (R 3.5.0)
##  bindrcpp     * 0.2.2    2018-03-29 [1] CRAN (R 3.5.0)
##  broom          0.5.1    2018-12-05 [1] CRAN (R 3.5.1)
##  callr          3.0.0    2018-08-24 [1] CRAN (R 3.5.0)
##  cli            1.0.1    2018-09-25 [1] CRAN (R 3.5.0)
##  colorspace     1.3-2    2016-12-14 [1] CRAN (R 3.5.0)
##  crayon         1.3.4    2017-09-16 [1] CRAN (R 3.5.0)
##  desc           1.2.0    2018-05-01 [1] CRAN (R 3.5.0)
##  devtools       2.0.1    2018-10-26 [1] CRAN (R 3.5.1)
##  digest         0.6.18   2018-10-10 [1] CRAN (R 3.5.0)
##  dplyr        * 0.7.8    2018-11-10 [1] CRAN (R 3.5.0)
##  evaluate       0.12     2018-10-09 [1] CRAN (R 3.5.0)
##  fansi          0.4.0    2018-10-05 [1] CRAN (R 3.5.0)
##  farver         1.1.0    2018-11-20 [1] CRAN (R 3.5.0)
##  forcats      * 0.3.0    2018-02-19 [1] CRAN (R 3.5.0)
##  fs             1.2.6    2018-08-23 [1] CRAN (R 3.5.0)
##  generics       0.0.2    2018-11-29 [1] CRAN (R 3.5.0)
##  ggforce        0.1.3    2018-07-07 [1] CRAN (R 3.5.0)
##  ggplot2      * 3.1.0    2018-10-25 [1] CRAN (R 3.5.0)
##  ggraph       * 1.0.2    2018-07-07 [1] CRAN (R 3.5.0)
##  ggrepel      * 0.8.0    2018-05-09 [1] CRAN (R 3.5.0)
##  glue           1.3.0    2018-07-17 [1] CRAN (R 3.5.0)
##  gridExtra      2.3      2017-09-09 [1] CRAN (R 3.5.0)
##  gtable         0.2.0    2016-02-26 [1] CRAN (R 3.5.0)
##  here         * 0.1      2017-05-28 [1] CRAN (R 3.5.0)
##  highr          0.7      2018-06-09 [1] CRAN (R 3.5.0)
##  htmltools      0.3.6    2017-04-28 [1] CRAN (R 3.5.0)
##  httr           1.3.1    2017-08-20 [1] CRAN (R 3.5.0)
##  igraph       * 1.2.2    2018-07-27 [1] CRAN (R 3.5.0)
##  janeaustenr    0.1.5    2017-06-10 [1] CRAN (R 3.5.0)
##  jsonlite       1.5      2017-06-01 [1] CRAN (R 3.5.0)
##  knitr        * 1.20     2018-02-20 [1] CRAN (R 3.5.0)
##  labeling       0.3      2014-08-23 [1] CRAN (R 3.5.0)
##  lattice        0.20-38  2018-11-04 [1] CRAN (R 3.5.1)
##  lazyeval       0.2.1    2017-10-29 [1] CRAN (R 3.5.0)
##  lubridate    * 1.7.4    2018-04-11 [1] CRAN (R 3.5.0)
##  magrittr       1.5      2014-11-22 [1] CRAN (R 3.5.0)
##  MASS           7.3-51.1 2018-11-01 [1] CRAN (R 3.5.0)
##  Matrix         1.2-15   2018-11-01 [1] CRAN (R 3.5.0)
##  memoise        1.1.0    2017-04-21 [1] CRAN (R 3.5.0)
##  modeltools     0.2-22   2018-07-16 [1] CRAN (R 3.5.0)
##  munsell        0.5.0    2018-06-12 [1] CRAN (R 3.5.0)
##  nlme           3.1-137  2018-04-07 [1] CRAN (R 3.5.1)
##  NLP            0.2-0    2018-10-18 [1] CRAN (R 3.5.0)
##  pillar         1.3.0    2018-07-14 [1] CRAN (R 3.5.0)
##  pkgbuild       1.0.2    2018-10-16 [1] CRAN (R 3.5.0)
##  pkgconfig      2.0.2    2018-08-16 [1] CRAN (R 3.5.0)
##  pkgload        1.0.2    2018-10-29 [1] CRAN (R 3.5.0)
##  plyr           1.8.4    2016-06-08 [1] CRAN (R 3.5.0)
##  prettyunits    1.0.2    2015-07-13 [1] CRAN (R 3.5.0)
##  processx       3.2.1    2018-12-05 [1] CRAN (R 3.5.1)
##  ps             1.2.1    2018-11-06 [1] CRAN (R 3.5.0)
##  purrr        * 0.2.5    2018-05-29 [1] CRAN (R 3.5.0)
##  R6             2.3.0    2018-10-04 [1] CRAN (R 3.5.0)
##  RColorBrewer * 1.1-2    2014-12-07 [1] CRAN (R 3.5.0)
##  Rcpp           1.0.0    2018-11-07 [1] CRAN (R 3.5.0)
##  remotes        2.0.2    2018-10-30 [1] CRAN (R 3.5.0)
##  reshape2       1.4.3    2017-12-11 [1] CRAN (R 3.5.0)
##  rlang          0.3.0.1  2018-10-25 [1] CRAN (R 3.5.0)
##  rmarkdown      1.10     2018-06-11 [1] CRAN (R 3.5.0)
##  rprojroot      1.3-2    2018-01-03 [1] CRAN (R 3.5.0)
##  rtweet       * 0.6.8    2018-09-28 [1] CRAN (R 3.5.0)
##  scales         1.0.0    2018-08-09 [1] CRAN (R 3.5.0)
##  sessioninfo    1.1.1    2018-11-05 [1] CRAN (R 3.5.0)
##  slam           0.1-43   2018-04-23 [1] CRAN (R 3.5.0)
##  SnowballC      0.5.1    2014-08-09 [1] CRAN (R 3.5.0)
##  stringi        1.2.4    2018-07-20 [1] CRAN (R 3.5.0)
##  stringr      * 1.3.1    2018-05-10 [1] CRAN (R 3.5.0)
##  testthat       2.0.1    2018-10-13 [1] CRAN (R 3.5.0)
##  tibble         1.4.2    2018-01-22 [1] CRAN (R 3.5.0)
##  tidyr        * 0.8.2    2018-10-28 [1] CRAN (R 3.5.0)
##  tidyselect     0.2.5    2018-10-11 [1] CRAN (R 3.5.0)
##  tidytext     * 0.2.0    2018-10-17 [1] CRAN (R 3.5.0)
##  tm             0.7-5    2018-07-29 [1] CRAN (R 3.5.0)
##  tokenizers     0.2.1    2018-03-29 [1] CRAN (R 3.5.0)
##  topicmodels  * 0.2-7    2017-11-03 [1] CRAN (R 3.5.0)
##  tweenr         1.0.0    2018-09-27 [1] CRAN (R 3.5.0)
##  units          0.6-2    2018-12-05 [1] CRAN (R 3.5.0)
##  usethis        1.4.0    2018-08-14 [1] CRAN (R 3.5.0)
##  utf8           1.1.4    2018-05-24 [1] CRAN (R 3.5.0)
##  viridis      * 0.5.1    2018-03-29 [1] CRAN (R 3.5.0)
##  viridisLite  * 0.3.0    2018-02-01 [1] CRAN (R 3.5.0)
##  withr          2.1.2    2018-03-15 [1] CRAN (R 3.5.0)
##  wordcloud    * 2.6      2018-08-24 [1] CRAN (R 3.5.0)
##  xml2           1.2.0    2018-01-24 [1] CRAN (R 3.5.0)
##  yaml           2.2.0    2018-07-25 [1] CRAN (R 3.5.0)
## 
## [1] /Library/Frameworks/R.framework/Versions/3.5/Resources/library</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
