<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Rick Tankard, modified from script by Luke Zappia" />


<title>Twitter coverage of the AMSI BioInfoSummer 2018 symposium in bioinformatics</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Conference Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="BioInfoSummer2018.html">BioInfoSummer2018</a>
    </li>
    <li>
      <a href="ABACBS18.html">ABACBS18</a>
    </li>
    <li>
      <a href="COMBINE18.html">COMBINE18</a>
    </li>
    <li>
      <a href="ASHG18.html">ASHG18</a>
    </li>
    <li>
      <a href="gi2018.html">GI2018</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/lazappi/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the AMSI BioInfoSummer 2018 symposium in bioinformatics</h1>
<h4 class="author"><em>Rick Tankard, modified from script by Luke Zappia</em></h4>
<h4 class="date"><em>2018-12-10 11:46:55</em></h4>

</div>


<p><strong>Parameters</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">Parameter</th>
<th align="left">Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Hashtag</td>
<td align="left">#bioinfosummer</td>
</tr>
<tr class="even">
<td align="left">Conference</td>
<td align="left">Bioinfosummer 2018</td>
</tr>
<tr class="odd">
<td align="left">Time Zone</td>
<td align="left">Australia/Perth</td>
</tr>
<tr class="even">
<td align="left">Days</td>
<td align="left">c(‚Äú2018-12-03‚Äù, ‚Äú2018-12-04‚Äù, ‚Äú2018-12-05‚Äù, ‚Äú2018-12-06‚Äù, ‚Äú2018-12-07‚Äù)</td>
</tr>
<tr class="odd">
<td align="left">Accent colour</td>
<td align="left">#dd0c15</td>
</tr>
<tr class="even">
<td align="left">Pallete</td>
<td align="left">Set1</td>
</tr>
<tr class="odd">
<td align="left">k-core</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">Topics k</td>
<td align="left">6</td>
</tr>
<tr class="odd">
<td align="left">Bigram filter</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">Fixed</td>
<td align="left">FALSE</td>
</tr>
</tbody>
</table>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Bioinfosummer 2018 conference. A total of 963 tweets from 184 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/Bioinfosummer2018/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2018-12-03 - 2018-12-07 in the Australia/Perth timezone.</p>
<p><img src="figures/Bioinfosummer2018/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/Bioinfosummer2018/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/Bioinfosummer2018/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/Bioinfosummer2018/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The ‚Äúreplies network‚Äù, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/Bioinfosummer2018/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The ‚Äúmentions network‚Äù, where users mention other users in their tweets. Filtered for a k-core of 2. Node colour and size adjusted according to PageRank score.</p>
<p><img src="figures/Bioinfosummer2018/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/Bioinfosummer2018/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/Bioinfosummer2018/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
My #BioInfoSummer links and slides are posted here (see slides link under other learning resources): <a href="https://t.co/FUF5xif5Zu" class="uri">https://t.co/FUF5xif5Zu</a> - see <a href="https://t.co/IDYudnx1RE" class="uri">https://t.co/IDYudnx1RE</a> for future updates to these materials or ask me if you need help finding/getting something
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Slides from my talk at #Bioinfosummer are now available online: <a href="https://t.co/kJqaAK6eNU" class="uri">https://t.co/kJqaAK6eNU</a> Keep an eye out for Thursday‚Äôs scMerge workshop materials! Really appreciate the invitation from organisers of <span class="citation">@Bioinfosummer</span> so glad to be involved üòäüòä
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Great talk by <span class="citation">@3rdreviewer</span> on how evolutionary analysis can fail when you assume the input genomes are correct. Doing a 1 year sabbatical at ANU. #BioInfoSummer <a href="https://t.co/wGcl1Qtrmm" class="uri">https://t.co/wGcl1Qtrmm</a>
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
rladiesperth
</td>
<td style="text-align:left;">
<p>Want to continue your journey with R after #BioInfoSummer 2018?</p>
<p>An #RLadies chapter is launching in Perth on Mon 10th Dec <span class="citation">@PerkinsComms</span> from 4:30pm.</p>
<p>To celebrate we are hosting talks by local R-ladies and networking üßÄü•Ç</p>
Register (RSVP Thurs 7th Dec): <a href="https://t.co/Vcl4mwwVm2" class="uri">https://t.co/Vcl4mwwVm2</a> <a href="https://t.co/Cnnh4hymRc" class="uri">https://t.co/Cnnh4hymRc</a>
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
PhilippBayer
</td>
<td style="text-align:left;">
Have to try this - AGOUTI to fix genome assemblies/annotations #BioInfoSummer <a href="https://t.co/lnbNNIqxzL" class="uri">https://t.co/lnbNNIqxzL</a>
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
#BioInfoSummer workshop material on analysis of #scRNAseq data is at <a href="https://t.co/QzWaGbiGyF" class="uri">https://t.co/QzWaGbiGyF</a> (kindly borrowed w/ permission from wonderful #scRNAseq course <span class="citation">@m_hemberg</span> <span class="citation">@wikiselev</span> <span class="citation">@talandrews</span> <span class="citation">@Jenni_Westoby</span> <span class="citation">@davisjmcc</span> <span class="citation">@marenbuettner</span> here <a href="https://t.co/b6VVpt5eX3" class="uri">https://t.co/b6VVpt5eX3</a>, shortened for time)
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
This gets more interesting‚Ä¶ and is making me hungry‚Ä¶ Under the topic of #phylogenomics <span class="citation">@torstenseemann</span> @ #BioInfoSummer <a href="https://t.co/vpYcVTaXRe" class="uri">https://t.co/vpYcVTaXRe</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
laura_boykin
</td>
<td style="text-align:left;">
‚ÄòImpact factors were invented because people forgot how to think‚Äô Professor Susan Wilson #BioInfoSummer I official love Susan.
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
Identifying differentially expressed genome-wide long range enhancers with #HiC and #diffHic‚Ä¶ presented by <span class="citation">@WEHI_research</span> legend Prof Gordon Smyth at #BioInfoSummer <a href="https://t.co/A8clQQJZgg" class="uri">https://t.co/A8clQQJZgg</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
PhilippBayer
</td>
<td style="text-align:left;">
.<span class="citation">@stephaniehicks</span> is hiring postdocs at postdocs at Johns Hopkins! #BioInfoSummer <a href="https://t.co/EycVfaujYk" class="uri">https://t.co/EycVfaujYk</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/Bioinfosummer2018/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/Bioinfosummer2018/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Now preparing for my #Bioinfosummer introductory workshop to genome assembly and annotation. Don‚Äôt forget to bring your meat computer üß† and a writing instrument üñäÔ∏è. <a href="https://t.co/Scp3MTnw7b" class="uri">https://t.co/Scp3MTnw7b</a>
</td>
<td style="text-align:right;">
77
</td>
</tr>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Great talk by <span class="citation">@3rdreviewer</span> on how evolutionary analysis can fail when you assume the input genomes are correct. Doing a 1 year sabbatical at ANU. #BioInfoSummer <a href="https://t.co/wGcl1Qtrmm" class="uri">https://t.co/wGcl1Qtrmm</a>
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Congratulations to the AMSI #BioInfoSummer 2018 poster winners: Dr Erika Duan <span class="citation">@latrobe</span> Jessica Cheng <span class="citation">@ScienceUWA</span> each won a $400 gift card And Gabriel Hauswirth <span class="citation">@ARMI_Monash</span> received the <span class="citation">@thermofisher</span> wet lab prize! <span class="citation">@ErikaDuan</span> <span class="citation">@Bioinfosummer</span> <a href="https://t.co/hNrPSyVBGN" class="uri">https://t.co/hNrPSyVBGN</a>
</td>
<td style="text-align:right;">
38
</td>
</tr>
<tr>
<td style="text-align:left;">
PhilippBayer
</td>
<td style="text-align:left;">
Have to try this - AGOUTI to fix genome assemblies/annotations #BioInfoSummer <a href="https://t.co/lnbNNIqxzL" class="uri">https://t.co/lnbNNIqxzL</a>
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
Identifying differentially expressed genome-wide long range enhancers with #HiC and #diffHic‚Ä¶ presented by <span class="citation">@WEHI_research</span> legend Prof Gordon Smyth at #BioInfoSummer <a href="https://t.co/A8clQQJZgg" class="uri">https://t.co/A8clQQJZgg</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Slides from my talk at #Bioinfosummer are now available online: <a href="https://t.co/kJqaAK6eNU" class="uri">https://t.co/kJqaAK6eNU</a> Keep an eye out for Thursday‚Äôs scMerge workshop materials! Really appreciate the invitation from organisers of <span class="citation">@Bioinfosummer</span> so glad to be involved üòäüòä
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
iamstaceyhong
</td>
<td style="text-align:left;">
Learning about de novo assembly and annotation with <span class="citation">@torstenseemann</span> through fun paper exercises ‚úÇÔ∏èüóí #BioInfoSummer #workshop #genomics #microbialgenomics #prokka <a href="https://t.co/s2shf8O3nD" class="uri">https://t.co/s2shf8O3nD</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
My #BioInfoSummer links and slides are posted here (see slides link under other learning resources): <a href="https://t.co/FUF5xif5Zu" class="uri">https://t.co/FUF5xif5Zu</a> - see <a href="https://t.co/IDYudnx1RE" class="uri">https://t.co/IDYudnx1RE</a> for future updates to these materials or ask me if you need help finding/getting something
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Treated to a stunning peacock display at <span class="citation">@uwanews</span> ahead of today‚Äôs single cell session at #bioinfosummer üòç <span class="citation">@Bioinfosummer</span> <a href="https://t.co/N2OB4zDfjh" class="uri">https://t.co/N2OB4zDfjh</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Grateful to be greeted in Perth for #BioInfoSummer by <span class="citation">@PhilippBayer</span>, <span class="citation">@JasonWilliamsNY</span>, Dave Edwards with food, beer and upgraded toiletries! <a href="https://t.co/xMSUasBMf0" class="uri">https://t.co/xMSUasBMf0</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/Bioinfosummer2018/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/Bioinfosummer2018/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
It‚Äôs a pleasure to highlight and present work by our fantastic <span class="citation">@sydneybioinfo</span> group at #Bioinfosummer <span class="citation">@LinYingxin</span> <span class="citation">@KevinWang009</span> Taiyun KIm <span class="citation">@TheEllisPatrick</span> <span class="citation">@PengyiYang82</span> <a href="https://t.co/8yianCIBTM" class="uri">https://t.co/8yianCIBTM</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
sydneybioinfo
</td>
<td style="text-align:left;">
Excellent thread showing the software and papers/preprints presented by <span class="citation">@shazanfar</span> about Single Cell research at #Bioinfosummer <a href="https://t.co/hBeDM9dOhd" class="uri">https://t.co/hBeDM9dOhd</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/Bioinfosummer2018/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Now preparing for my #Bioinfosummer introductory workshop to genome assembly and annotation. Don‚Äôt forget to bring your meat computer üß† and a writing instrument üñäÔ∏è. <a href="https://t.co/Scp3MTnw7b" class="uri">https://t.co/Scp3MTnw7b</a>
</td>
<td style="text-align:right;">
77
</td>
</tr>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Great talk by <span class="citation">@3rdreviewer</span> on how evolutionary analysis can fail when you assume the input genomes are correct. Doing a 1 year sabbatical at ANU. #BioInfoSummer <a href="https://t.co/wGcl1Qtrmm" class="uri">https://t.co/wGcl1Qtrmm</a>
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Congratulations to the AMSI #BioInfoSummer 2018 poster winners: Dr Erika Duan <span class="citation">@latrobe</span> Jessica Cheng <span class="citation">@ScienceUWA</span> each won a $400 gift card And Gabriel Hauswirth <span class="citation">@ARMI_Monash</span> received the <span class="citation">@thermofisher</span> wet lab prize! <span class="citation">@ErikaDuan</span> <span class="citation">@Bioinfosummer</span> <a href="https://t.co/hNrPSyVBGN" class="uri">https://t.co/hNrPSyVBGN</a>
</td>
<td style="text-align:right;">
38
</td>
</tr>
<tr>
<td style="text-align:left;">
PhilippBayer
</td>
<td style="text-align:left;">
Have to try this - AGOUTI to fix genome assemblies/annotations #BioInfoSummer <a href="https://t.co/lnbNNIqxzL" class="uri">https://t.co/lnbNNIqxzL</a>
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
Identifying differentially expressed genome-wide long range enhancers with #HiC and #diffHic‚Ä¶ presented by <span class="citation">@WEHI_research</span> legend Prof Gordon Smyth at #BioInfoSummer <a href="https://t.co/A8clQQJZgg" class="uri">https://t.co/A8clQQJZgg</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
iamstaceyhong
</td>
<td style="text-align:left;">
Learning about de novo assembly and annotation with <span class="citation">@torstenseemann</span> through fun paper exercises ‚úÇÔ∏èüóí #BioInfoSummer #workshop #genomics #microbialgenomics #prokka <a href="https://t.co/s2shf8O3nD" class="uri">https://t.co/s2shf8O3nD</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Treated to a stunning peacock display at <span class="citation">@uwanews</span> ahead of today‚Äôs single cell session at #bioinfosummer üòç <span class="citation">@Bioinfosummer</span> <a href="https://t.co/N2OB4zDfjh" class="uri">https://t.co/N2OB4zDfjh</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
torstenseemann
</td>
<td style="text-align:left;">
Grateful to be greeted in Perth for #BioInfoSummer by <span class="citation">@PhilippBayer</span>, <span class="citation">@JasonWilliamsNY</span>, Dave Edwards with food, beer and upgraded toiletries! <a href="https://t.co/xMSUasBMf0" class="uri">https://t.co/xMSUasBMf0</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
In beautiful Perth ahead of #BioInfoSummer this week üòä‚úå <a href="https://t.co/hIYpCqY6mx" class="uri">https://t.co/hIYpCqY6mx</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
This gets more interesting‚Ä¶ and is making me hungry‚Ä¶ Under the topic of #phylogenomics <span class="citation">@torstenseemann</span> @ #BioInfoSummer <a href="https://t.co/vpYcVTaXRe" class="uri">https://t.co/vpYcVTaXRe</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/Dtn1OUFVsAE9RUN.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/Bioinfosummer2018/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 3 times.</p>
<p><img src="figures/Bioinfosummer2018/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/Bioinfosummer2018/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Congratulations to the AMSI #BioInfoSummer 2018 poster winners: Dr Erika Duan <span class="citation">@latrobe</span> Jessica Cheng <span class="citation">@ScienceUWA</span> each won a $400 gift card And Gabriel Hauswirth <span class="citation">@ARMI_Monash</span> received the <span class="citation">@thermofisher</span> wet lab prize! <span class="citation">@ErikaDuan</span> <span class="citation">@Bioinfosummer</span> <a href="https://t.co/hNrPSyVBGN" class="uri">https://t.co/hNrPSyVBGN</a>
</td>
<td style="text-align:right;">
0.9945614
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
Lim_SuChen
</td>
<td style="text-align:left;">
<ol start="4" style="list-style-type: decimal">
<li>Tomorrow, I‚Äôll be meeting <span class="citation">@shihchingfu</span> to talk about how to publish a booküòÑ. He and his scientist wife published a book that aims to inspire young children to do science. Super excited as I‚Äôm thinking of illustrating a book myself as a side gig. #BioInfoSummer Networkingüëç <a href="https://t.co/YlkmQGGqfR" class="uri">https://t.co/YlkmQGGqfR</a>
</td>
<td style="text-align:right;">
0.9940471
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
Professor Wilson‚Äôs slides are so relevant to the #BioInfoSummer audience! If my slides weren‚Äôt already written I could have used these! I suspect all the speakers will be converging on these key themes that represent our best approach to sound scientific practice. <a href="https://t.co/woIViKvl69" class="uri">https://t.co/woIViKvl69</a>
</td>
<td style="text-align:right;">
0.9934254
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
#BioInfoSummer workshop material on analysis of #scRNAseq data is at <a href="https://t.co/QzWaGbiGyF" class="uri">https://t.co/QzWaGbiGyF</a> (kindly borrowed w/ permission from wonderful #scRNAseq course <span class="citation">@m_hemberg</span> <span class="citation">@wikiselev</span> <span class="citation">@talandrews</span> <span class="citation">@Jenni_Westoby</span> <span class="citation">@davisjmcc</span> <span class="citation">@marenbuettner</span> here <a href="https://t.co/b6VVpt5eX3" class="uri">https://t.co/b6VVpt5eX3</a>, shortened for time)
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Next, As. Prof Stephanie Hicks takes us through the scRNAseq workflow via Bioconductor. Currently over 58 packages available for scRNAseq analysis. ScRNAseq analysis has additional noise parameters that need to be considered. #BioInfoSummer <a href="https://t.co/RJAONRqUPF" class="uri">https://t.co/RJAONRqUPF</a>
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
Lovely walk by the water üèñ &amp;amp; now ready for #BioInfoSummer Day 2 on #singlecell #omics üë©üèæ‚Äçüíªüë®üèª‚Äçüíªüë©‚Äçüíªüë®üèº‚Äçüíªüë®üèø‚Äçüíªüë©üèª‚Äçüíªto kickoff <span class="citation">@uwanews</span> (even the peacocks ü¶ö seem to be ready!) #Bioinformatics #scRNAseq #statistics #mathematics #STEM <a href="https://t.co/aRBcNptw3d" class="uri">https://t.co/aRBcNptw3d</a>
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Sad to miss #ASI2018WA but so excited to be attending #BioInfoSummer <span class="citation">@Bioinfosummer</span>. A poor understanding of statistics, especially in biology, is a contributor to poor science reproducibility. <a href="https://t.co/viauNeCM6k" class="uri">https://t.co/viauNeCM6k</a>
</td>
<td style="text-align:right;">
0.9911024
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
<span class="citation">@AswinSSoman</span> <span class="citation">@m_hemberg</span> <span class="citation">@wikiselev</span> <span class="citation">@talandrews</span> <span class="citation">@Jenni_Westoby</span> <span class="citation">@davisjmcc</span> <span class="citation">@marenbuettner</span> Thanks! I believe the original course material is updated twice a year at a course that is taught in the UK, so should be updated soon! The course material for #BioInfoSummer was updated to most recent advancements in 2018.
</td>
<td style="text-align:right;">
0.9911024
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
shihchingfu
</td>
<td style="text-align:left;">
Introductory workshop to metabolomics and mass spectrometry by Dr Joel Gummer from <span class="citation">@MetabolomicsMU</span> <span class="citation">@MurdochUni</span>. Had to dust off my #msexcel skills for my last workshop at #BioInfoSummer 2018. <a href="https://t.co/xprPxGVTjn" class="uri">https://t.co/xprPxGVTjn</a>
</td>
<td style="text-align:right;">
0.9911024
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
My #BioInfoSummer links and slides are posted here (see slides link under other learning resources): <a href="https://t.co/FUF5xif5Zu" class="uri">https://t.co/FUF5xif5Zu</a> - see <a href="https://t.co/IDYudnx1RE" class="uri">https://t.co/IDYudnx1RE</a> for future updates to these materials or ask me if you need help finding/getting something
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Slides from my talk at #Bioinfosummer are now available online: <a href="https://t.co/kJqaAK6eNU" class="uri">https://t.co/kJqaAK6eNU</a> Keep an eye out for Thursday‚Äôs scMerge workshop materials! Really appreciate the invitation from organisers of <span class="citation">@Bioinfosummer</span> so glad to be involved üòäüòä
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Join Prof.¬†Gordon Smyth from <span class="citation">@WEHI_research</span> as he introduces Hi-C technology and outlines a bioinformatics analysis approach using R and Bioconductor software next at #BioInfoSummer
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
Bioinfosummer
</td>
<td style="text-align:left;">
AMSI #BioInfoSummer best poster winners will be announced at the <span class="citation">@Bioinfosummer</span> closing ceremony. Make sure you come back to the Alexander Lecture Theatre after the workshop session for the conference wrap up.
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ARMI_Labs
</td>
<td style="text-align:left;">
Congratulations to <span class="citation">@DevBioBoy</span>, a PhD student with the McGlinn group, on his recent prize at the 2018 <span class="citation">@DiscoverAMSI</span> #bioinfosummer program and symposium for bioinformatics and computational biology. Well done Gabriel! <a href="https://t.co/yBvmeQ9RgI" class="uri">https://t.co/yBvmeQ9RgI</a>
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
</tbody>
</table></li>
</ol>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#BioInfoSummer <span class="citation">@Al__Forrest</span> describing ways to tackle understanding of cell-cell communication networks, using expression of ligand and receptor genes in single cells and cell types. Is there a high quality database of ligand-receptor gene networks?
</td>
<td style="text-align:right;">
0.9940471
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#BioInfoSummer <span class="citation">@Al__Forrest</span> describing reframing Q of annotation of cell types. Instead of unsupervised learning (clustering) followed by labelling, start with reference profiles &amp;amp; perform supervised learning (classification) of cells into types. scMatch: <a href="https://t.co/dQB424Fcma" class="uri">https://t.co/dQB424Fcma</a>
</td>
<td style="text-align:right;">
0.9937517
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Prof Alistair Forrest begins day 2 of #BioInfoSummer introducing the WA cancer scRNAseq consortium. Lots of work and consideration for using reference expression profiles to annotate single cell clusters. <a href="https://t.co/VT2tBw5uo5" class="uri">https://t.co/VT2tBw5uo5</a>
</td>
<td style="text-align:right;">
0.9937517
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
<span class="citation">@JasonWilliamsNY</span> improving the bioinformatics curriculum,including prac projects in undergraduate courses would be ideal- I couldn‚Äôt agree more, my third year practical project in my degree shaped my path in science for sure! <span class="citation">@Bioinfosummer</span> <span class="citation">@DiscoverAMSI</span> #Bioinfosummer <a href="https://t.co/IQToPJBPri" class="uri">https://t.co/IQToPJBPri</a>
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
First #BioInfoSummer lecture of the day starting soon! Are you ready to learn about the Western Australian Cancer Single-Cell Initiative with <span class="citation">@Al__Forrest</span>, Head of the <span class="citation">@PerkinsComms</span> Systems Biology and Genomics Lab?
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#bioinfosummer <span class="citation">@stephaniehicks</span> speaking on ‚ÄòOrchestrating Single-Cell RNA-Sequencing Analysis with R/Bioconductor‚Äô co-founder of <span class="citation">@RLadiesBmore</span> and fun-fact Stephanie is writing a children‚Äôs book about data science!
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
<span class="citation">@stephaniehicks</span> speaking on single cell RNA seq and a bit about Bioconductor this morning - feeling happy about the intro to R workshop I attended yesterday afternoon! Looking forward to trying some of this out later <span class="citation">@DiscoverAMSI</span> <span class="citation">@Bioinfosummer</span> #BioinfoSummer <a href="https://t.co/99gIHdOgQZ" class="uri">https://t.co/99gIHdOgQZ</a>
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
In the #BioInfoSummer opening lecture by Professor Susan Wilson, University of NSW puts a fine point of the difference between reproducibility and replicability - we want to be able to replicate findings and often a better understanding of statistics is what‚Äôs needed! <a href="https://t.co/yt3AbIpSVZ" class="uri">https://t.co/yt3AbIpSVZ</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#Bioinfosummer <span class="citation">@JasonWilliamsNY</span> gives a reality check. Learning doesn‚Äôt happen <em>AT</em> the workshop, it happens <em>AFTER</em> the workshop. That‚Äôs why ensuring resources are available continuously as well as people + networks (e.g. #rladies <span class="citation">@RLadiesSydney</span> <span class="citation">@combine_au</span> <span class="citation">@abacbs</span> )
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
<span class="citation">@Al__Forrest</span> first up this morning for #BioinfoSummer with discussion of single cell genomics and the Western Australian Cancer Single Cell Initiative <span class="citation">@PerkinsComms</span> <span class="citation">@DiscoverAMSI</span> <span class="citation">@Bioinfosummer</span> <a href="https://t.co/1ZZauGBBaE" class="uri">https://t.co/1ZZauGBBaE</a>
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
As someone who didn‚Äôt receive any undergraduate <em>bioinformatics</em> training, I feel it‚Äôs so important to think about the need for building bioinformatics into curriculum <span class="citation">@JasonWilliamsNY</span> events like #Bioinfosummer bridge this gap by bringing many resources together
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
<span class="citation">@svheeringen</span> on the importance of the cis-regulatory grammar underying cell fate choice to stem cell research and regenerative medicine #BioInfoSummer
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
<p>Nice FF talk by <span class="citation">@LCrowhurstIMB</span> (working with <span class="citation">@drjosephpowell</span> ) describing scIVA for single cell interactive visualisation and analysis</p>
<p><a href="https://t.co/UZ8c67lrDe" class="uri">https://t.co/UZ8c67lrDe</a></p>
<p><a href="https://t.co/yDCXdTDF6j" class="uri">https://t.co/yDCXdTDF6j</a></p>
#Bioinfosummer
</td>
<td style="text-align:right;">
0.9904259
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Prof Inge Koch hosting the <span class="citation">@DiscoverAMSI</span> CHOOSEMATHS Lunch at #BioInfoSummer programs include encouraging yr 9-10 high school students to take senior high school maths. Aus-wide stats show underrepresentation exists at the high school level and poor attitudes even earlier (yr 5) <a href="https://t.co/P59CP6y6wL" class="uri">https://t.co/P59CP6y6wL</a>
</td>
<td style="text-align:right;">
0.9953626
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Prof Rebecca Johnson from the Australian Museum gives a fascinating talk about an accidental career in wildlife forensics. Wildlife crime has a huge cost across the world and smuggling has especially detrimental impact on island nations (pests introduction etc). #BioInfoSummer <a href="https://t.co/B0J8kA1WYq" class="uri">https://t.co/B0J8kA1WYq</a>
</td>
<td style="text-align:right;">
0.9947866
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
rladiesperth
</td>
<td style="text-align:left;">
<p>Want to continue your journey with R after #BioInfoSummer 2018?</p>
<p>An #RLadies chapter is launching in Perth on Mon 10th Dec <span class="citation">@PerkinsComms</span> from 4:30pm.</p>
<p>To celebrate we are hosting talks by local R-ladies and networking üßÄü•Ç</p>
Register (RSVP Thurs 7th Dec): <a href="https://t.co/Vcl4mwwVm2" class="uri">https://t.co/Vcl4mwwVm2</a> <a href="https://t.co/Cnnh4hymRc" class="uri">https://t.co/Cnnh4hymRc</a>
</td>
<td style="text-align:right;">
0.9947866
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Prof Dave Edwards giving an excellent talk on building pan genomes in crop plants. A number of morphotypically different plants have very similar/same genomes! Diverse genes tend to be bacterial defence-related #BioInfoSummer <a href="https://t.co/YBThd6fzXl" class="uri">https://t.co/YBThd6fzXl</a>
</td>
<td style="text-align:right;">
0.9943158
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
Prof Gordon Smyth on using Hi-C to explore chromatin interactions and architecture across the genome on promoter-enhancer loops, TADs and chromosome (A/B) levels. diffHC detects differential interactions across biological conditions #BioInfoSummer <a href="https://t.co/RzOsFlTRch" class="uri">https://t.co/RzOsFlTRch</a>
</td>
<td style="text-align:right;">
0.9940471
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@ryanlisterlab</span> starting off day 4 on the importance of understanding the epigenome and epigenetic changes underlying normal and disease development - interrogated by technologies, such as <span class="citation">@nanopore</span> - advs include long reads and no reliance on bisulphite conversion #BioInfoSummer <a href="https://t.co/Cij2EhE4rf" class="uri">https://t.co/Cij2EhE4rf</a>
</td>
<td style="text-align:right;">
0.9940471
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
The UK‚Äôs got the black üêë‚Ä¶ ü§≠üòÖ Dr.¬†Marina Naval Sanchez from #CSIRO explains how functional annotation reveals proximal regulatory elements contributed to the evolution of modern sheep breeds #BioInfoSummer <a href="https://t.co/Jl45XDkdaq" class="uri">https://t.co/Jl45XDkdaq</a>
</td>
<td style="text-align:right;">
0.9937517
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
Conference opening by Peter Klinken AC, Chief Scientist of Western Australia, with the statement ‚Äúthink outside the box‚Ä¶ not just as a bioinformatician but a think how your capabilities are applicable in a wide veriety of different areas‚Äù #BioInfoSummer 2018 <a href="https://t.co/YBBunOI4Os" class="uri">https://t.co/YBBunOI4Os</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Prof Dave Edwards has a great analogy for genome sequencing challenges. How good the alignment is, whether SNPs can be linked to phenotypes and building pangenomes are important for plant genomics. #BioinfoSummer <a href="https://t.co/L0KepDj6v0" class="uri">https://t.co/L0KepDj6v0</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
#BioInfoSummer job seekers! St Vincent‚Äôs Hospital Melbourne needs an experienced #biostatistician to join the Australian Scleroderma Interest Group (ASIG) - apply before Monday 17 Dec at <a href="https://t.co/x0vRP2wvCk" class="uri">https://t.co/x0vRP2wvCk</a> <span class="citation">@Bioinfosummer</span>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
<span class="citation">@shazanfar</span> introducing scMerge, which allows integration of multiple datasets - scMerge battles strong batch effects between datasets by estimating unwanted variation using negative controls and replicates. Featuring a surprise appearance by Drake #BioInfoSummer <a href="https://t.co/iRVQnnZnr9" class="uri">https://t.co/iRVQnnZnr9</a>
</td>
<td style="text-align:right;">
0.9943158
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shihchingfu
</td>
<td style="text-align:left;">
Last day of #BioInfoSummer. Been a great week and we‚Äôll finish strong on metabolomics and proteomics with Dr Camilla Hill, Dr Timo Lassmann, Dr Ashley Waardenberg, Dr Stacey Reinke, and Dr Joel Gummer. <span class="citation">@DiscoverAMSI</span>
</td>
<td style="text-align:right;">
0.9940471
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
#BioInfoSummer <span class="citation">@stephaniehicks</span> excellent framing of the issues of batch effects in scRNA-Seq data, I‚Äôm so glad the audience is primed to hear about more scRNA-Seq and scMerge after morning tea! <a href="https://t.co/ykSvb5w4UR" class="uri">https://t.co/ykSvb5w4UR</a>
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
I always enjoy talks by <span class="citation">@torstenseemann</span> - doesn‚Äôt disappoint today, great discussion about bacteria, wgs, genomics and bioinformatics for public health. I can only hope the plant pathogens can follow suit ASAP! #BioInfoSummer <span class="citation">@Bioinfosummer</span> <span class="citation">@DiscoverAMSI</span> <a href="https://t.co/DQE06UFroi" class="uri">https://t.co/DQE06UFroi</a>
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
Dr.¬†Camilla Hill starting off day 5 on reduced representation sequencing strategies (such as solution-capture) and bioinformatics (GYDLE) in understanding and maximizing yield in barley #BioInfoSummer
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
First up today <span class="citation">@Bioinfosummer</span> is Dr Camilla Hill from <span class="citation">@MurdochUni</span> with her work on Barley Genomics to maximise yield as part of the Western Barley Genetics Alliance - great collaboration on an important crop with <span class="citation">@DPIRDbroadacre</span> and <span class="citation">@theGRDC</span>. #BioInfoSummer <span class="citation">@DiscoverAMSI</span> <a href="https://t.co/VGvjgx6qlV" class="uri">https://t.co/VGvjgx6qlV</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shihchingfu
</td>
<td style="text-align:left;">
Dr Ashley Waardenberg from <span class="citation">@CTBMB_JCU</span> (who was on the <span class="citation">@combine_au</span> panel last night) speaking on kinase activity and statistical inference using KinSwingR. Food for thought: perhaps we need to be looking at the sensor rather than fixating on the lens. #BioInfoSummer <span class="citation">@DiscoverAMSI</span> <a href="https://t.co/V2WS35Jgpf" class="uri">https://t.co/V2WS35Jgpf</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
shihchingfu
</td>
<td style="text-align:left;">
Dr Camilla Hill, Another local scientist doing great work on barley at Western Barley Genetics Alliance; a collaboration between <span class="citation">@MurdochUni</span> and Dept of Primary Industry. #BioInfoSummer <a href="https://t.co/jnPTadb42h" class="uri">https://t.co/jnPTadb42h</a>
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@AWaardenberg</span> on phosphoproteomics - most CNS diseases act at the synapse, and neurotransmission is largely phosphoregulated - phosphoProcessR is used in the workflow, with KinSwingR used to predict kinase activity at a particular peptide #BioInfoSummer <a href="https://t.co/CmcWbDAG9W" class="uri">https://t.co/CmcWbDAG9W</a>
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Everybody gets a sleep in on Day 4 of <span class="citation">@DiscoverAMSI</span> #BioInfoSummer 2018 - the first session starts at 9.30am! Remember to view the workshop pages so you are prepared for the afternoon sessions. <a href="https://t.co/zGGqgw4wpX" class="uri">https://t.co/zGGqgw4wpX</a> <a href="https://t.co/ZqgxC4FuJZ" class="uri">https://t.co/ZqgxC4FuJZ</a>
</td>
<td style="text-align:right;">
0.9911024
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Meet AMSI #BioInfoSummer 2018 speaker Marina Naval Sanchez from <span class="citation">@CSIROnews</span>! Join Marina to learn about molecular processes driving the evolutionary changes produced by sheep domestication and artificial selection <a href="https://t.co/I0SwiH4Rlz" class="uri">https://t.co/I0SwiH4Rlz</a> <span class="citation">@UWAresearch</span> <span class="citation">@VetLifeScience</span> <span class="citation">@ECUresearch</span> <a href="https://t.co/ZH3S4sXRzd" class="uri">https://t.co/ZH3S4sXRzd</a>
</td>
<td style="text-align:right;">
0.9947866
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
JasonWilliamsNY
</td>
<td style="text-align:left;">
Super excited to listen to <span class="citation">@DrRebeccaJ</span> - the first woman director of research in the <span class="citation">@austmus</span> ‚Äòs 191 year history. She is a true Superstar and a Superstar woman in STEM. Tonight she is sharing her science and her inspiring journey. #BioInfoSummer - look her up, follow her! <a href="https://t.co/kqG2NTeJY8" class="uri">https://t.co/kqG2NTeJY8</a>
</td>
<td style="text-align:right;">
0.9937517
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
shazanfar
</td>
<td style="text-align:left;">
Enjoying the morning session of #BioInfoSummer with Sue Wilson presenting the Opening Lecture. Questions touching on keeping track of methods (e.g.¬†software versions) to ensure reproducibility. The person you collaborate with the <em>most</em> is future you üòÖ
</td>
<td style="text-align:right;">
0.9934254
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
rladiesperth
</td>
<td style="text-align:left;">
<p>Don‚Äôt forget to register for our #RLadies Perth launch event. Registrations close tomorrow night. We are looking forward to meeting you all! üéâüéâ</p>
#BioInfoSummer <span class="citation">@ECUresearch</span> <span class="citation">@MurdochUni</span> <span class="citation">@CurtinIC</span> <span class="citation">@CurtinUni</span> <span class="citation">@UWAresearch</span> <span class="citation">@BiolSci_UWA</span> <span class="citation">@telethonkids</span> <span class="citation">@PerkinsComms</span> <a href="https://t.co/aGh41pqjQ4" class="uri">https://t.co/aGh41pqjQ4</a>
</td>
<td style="text-align:right;">
0.9934254
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
shihchingfu
</td>
<td style="text-align:left;">
Dr Timo Lassmann of <span class="citation">@telethonkids</span> sharing at #BioInfoSummer about applying infinite Hidden Markov models to sequencing. Although not trained to find motifs his HMM finds them! Also, he‚Äôs a fellow Emacs user #editorwars <a href="https://t.co/20l2WLEIEw" class="uri">https://t.co/20l2WLEIEw</a>
</td>
<td style="text-align:right;">
0.9934254
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
In the mood for a catered lunch? If you‚Äôre a #BioInfoSummer attendee, join us on the Arts Building Lawns now! Female attendees can also head to the Arts Building Seminar Room for a #CHOOSEMATHS lunch event <span class="citation">@UWAresearch</span>
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
Bioinfosummer
</td>
<td style="text-align:left;">
FREE <span class="citation">@DiscoverAMSI</span> #BioInfoSummer 2018 public lecture by <span class="citation">@austmus</span> Director Rebecca Johnson. Accessible to year 12 students and above. Register at <a href="https://t.co/AqIrCqioVT" class="uri">https://t.co/AqIrCqioVT</a> <span class="citation">@MAWAinc</span> <span class="citation">@ScitechWA</span> <span class="citation">@ScienceUWA</span> <span class="citation">@UWAresearch</span> <span class="citation">@ECU</span> <span class="citation">@MurdochUni</span> <span class="citation">@CurtinUni</span> <span class="citation">@DrRebeccaJ</span> <a href="https://t.co/H3t1UGe6hv" class="uri">https://t.co/H3t1UGe6hv</a>
</td>
<td style="text-align:right;">
0.9930632
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Have you registered yet for the free AMSI #BioInfoSummer 2018 Public Lecture? Wed 5 Dec at 6pm <span class="citation">@UWABusSchool</span>! <a href="https://t.co/pJuzWAO2YU" class="uri">https://t.co/pJuzWAO2YU</a> <span class="citation">@Bioinfosummer</span> <span class="citation">@ausbionet</span> <span class="citation">@VetLifeScience</span> <span class="citation">@UWAresearch</span> <span class="citation">@CurtinUni</span> <span class="citation">@ECUresearch</span> <span class="citation">@auscicomm</span> <a href="https://t.co/EmJxCHunJt" class="uri">https://t.co/EmJxCHunJt</a>
</td>
<td style="text-align:right;">
0.9926588
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
.<span class="citation">@JasonWilliamsNY</span> one of the biggest barriers is lack of faculty expertise / training and lack of student background skills / knowledge. Pretty consistent across the type of insitution too #BioInfoSummer <a href="https://t.co/LlZ0rYRUlV" class="uri">https://t.co/LlZ0rYRUlV</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
What a night! Great hearing from our insightful speakers <span class="citation">@monica_kehoe</span> <span class="citation">@AWaardenberg</span> <span class="citation">@3rdreviewer</span> &amp;amp; Prof Sue Wilson on the many bioinformatics careers paths available. Thanks <span class="citation">@combine_au</span> for hosting the AMSI #BIoInfoSummer 2018 Careers Evening.
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
urwahnawaz
</td>
<td style="text-align:left;">
Dr Timo Lassmann talking about modelling biological sequences using Hidden Markov models. I‚Äôm loving his #beamer theme! #BioInfoSummer #LaTeX <a href="https://t.co/KP7YldyQgb" class="uri">https://t.co/KP7YldyQgb</a>
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:right;">
topic
</th>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@laura_boykin</span> on portable sequencing to diagnose unhealthy Cassava for farmers in East Africa - delivering results to farmers in real time (Tree Lab). 800 million people in the world are dependent on Cassava for daily calories -very inspiring story!! #BioInfoSummer <a href="https://t.co/a0FY8P2NNL" class="uri">https://t.co/a0FY8P2NNL</a>
</td>
<td style="text-align:right;">
0.9949938
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@ZhipingWeng</span> starting off day 3 on the sequencing, assembly and annotation of the Hi5 cell line from Trichoplusia ni (the cabbage looper!), which is a great in vitro system to study small RNAs such as piRNAs. piggyBac transposons were originally found in Hi5! #BioInfoSummer <a href="https://t.co/vKKgpOWB2F" class="uri">https://t.co/vKKgpOWB2F</a>
</td>
<td style="text-align:right;">
0.9947866
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@StaceyReinke</span> closing #BioInfoSummer talks with Data Integration - there are 3 types 1. Correlation-based; 2. pathway-based; and 3. multivariate projection methods (such as multi-block). The latter method reduces feature space and doesn‚Äôt require a priori biological knowledge
</td>
<td style="text-align:right;">
0.9947866
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
dilys_lam
</td>
<td style="text-align:left;">
.<span class="citation">@3rdreviewer</span> on the importance of error estimates and correction in plant and animal genome assemblies in evolutionary analysis -&amp;gt; error-prone base calling and gene numbers, with bad assemblies adding or removing genes #BioInfoSummer
</td>
<td style="text-align:right;">
0.9943158
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Prof Ryan Lister giving a clear overview of whole genome bisulfite sequencing on day 4 of #BioInfoSummer - one challenge is to be able to identify all functionally significant methylation sites from such a vast amount of data obtained. <a href="https://t.co/VFptlDbn38" class="uri">https://t.co/VFptlDbn38</a>
</td>
<td style="text-align:right;">
0.9937517
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
ErikaDuan
</td>
<td style="text-align:left;">
Next, A/Prof Simon van Herrigen talks about the holy grail for the H3k27 trimethylation field - how does PRC2 get recruited to DNA to put down these histone marks? #BioInfoSummer <a href="https://t.co/VtiTVaBGAN" class="uri">https://t.co/VtiTVaBGAN</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
DiscoverAMSI
</td>
<td style="text-align:left;">
Another great day of presentations at AMSI #BioInfoSummer 2018 topped off with an educational afternoon of workshops by two of our international speakers <span class="citation">@JasonWilliamsNY</span> and <span class="citation">@stephaniehicks</span>. Enjoy a sundowner tonight. See you at 9am tomorrow for <span class="citation">@Bioinfosummer</span> Day 3 <a href="https://t.co/BL80a3Skny" class="uri">https://t.co/BL80a3Skny</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
monica_kehoe
</td>
<td style="text-align:left;">
<span class="citation">@laura_boykin</span> fresh off the plane from East Africa sharing her experiences from #poresafari and #treelab with everyone <span class="citation">@Bioinfosummer</span> <span class="citation">@DiscoverAMSI</span> #BioInfoSummer <span class="citation">@nanopore</span> <span class="citation">@ScienceUWA</span> watch this space for exciting things to come and see <a href="https://t.co/9tusvgC2oP" class="uri">https://t.co/9tusvgC2oP</a> for more info <a href="https://t.co/4r1bidM1F6" class="uri">https://t.co/4r1bidM1F6</a>
</td>
<td style="text-align:right;">
0.9922042
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
stephaniehicks
</td>
<td style="text-align:left;">
Attending #BioInfoSummer in Perth, Australia this week! Already fell in love with the beautiful weather, beaches, and ping pong tables on the <span class="citation">@uwanews</span> campus üòª <a href="https://t.co/7saESF2HU9" class="uri">https://t.co/7saESF2HU9</a>
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
shaniAmare
</td>
<td style="text-align:left;">
Identifying differentially expressed genome-wide long range enhancers with #HiC and #diffHic‚Ä¶ presented by <span class="citation">@WEHI_research</span> legend Prof Gordon Smyth at #BioInfoSummer <a href="https://t.co/A8clQQJZgg" class="uri">https://t.co/A8clQQJZgg</a>
</td>
<td style="text-align:right;">
0.9916897
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="software" class="section level1">
<h1><span class="header-section-number">10</span> Software</h1>
<p>Software mentioned in Tweets with links to GitHub, BitBucket, Bioconductor or CRAN.</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="left">Type</th>
<th align="left">Link</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">bio-workshop-18</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/pawseysc/bio-workshop-18" class="uri">https://github.com/pawseysc/bio-workshop-18</a></td>
</tr>
<tr class="even">
<td align="left">bioinfosummer</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/simonvh/bioinfosummer" class="uri">https://github.com/simonvh/bioinfosummer</a></td>
</tr>
<tr class="odd">
<td align="left">fluff</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/simonvh/fluff" class="uri">https://github.com/simonvh/fluff</a></td>
</tr>
<tr class="even">
<td align="left">genomepy</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/simonvh/genomepy" class="uri">https://github.com/simonvh/genomepy</a></td>
</tr>
<tr class="odd">
<td align="left">gimmemotifs</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/vanheeringen-lab/gimmemotifs" class="uri">https://github.com/vanheeringen-lab/gimmemotifs</a></td>
</tr>
<tr class="even">
<td align="left">KinSwingR</td>
<td align="left">Bioconductor</td>
<td align="left"><a href="https://bioconductor.org/packages/KinSwingR" class="uri">https://bioconductor.org/packages/KinSwingR</a></td>
</tr>
<tr class="odd">
<td align="left">phosphoProcessR</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/awaardenberg/phosphoprocessr" class="uri">https://github.com/awaardenberg/phosphoprocessr</a></td>
</tr>
<tr class="even">
<td align="left">scIVA</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/imb-computational-genomics-lab/sciva" class="uri">https://github.com/imb-computational-genomics-lab/sciva</a></td>
</tr>
<tr class="odd">
<td align="left">scmatch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/forrest-lab/scmatch" class="uri">https://github.com/forrest-lab/scmatch</a></td>
</tr>
<tr class="even">
<td align="left">scMatch</td>
<td align="left">GitHub</td>
<td align="left"><a href="https://github.com/asrhou/scmatch" class="uri">https://github.com/asrhou/scmatch</a></td>
</tr>
</tbody>
</table>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## ‚îÄ Session info ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  setting  value                       
##  version  R version 3.5.1 (2018-07-02)
##  os       macOS High Sierra 10.13.6   
##  system   x86_64, darwin15.6.0        
##  ui       X11                         
##  language (EN)                        
##  collate  en_AU.UTF-8                 
##  ctype    en_AU.UTF-8                 
##  tz       Australia/Perth             
##  date     2018-12-10                  
## 
## ‚îÄ Packages ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ
##  package      * version  date       lib source        
##  assertthat     0.2.0    2017-04-11 [1] CRAN (R 3.5.0)
##  backports      1.1.2    2017-12-13 [1] CRAN (R 3.5.0)
##  base64enc      0.1-3    2015-07-28 [1] CRAN (R 3.5.0)
##  bindr          0.1.1    2018-03-13 [1] CRAN (R 3.5.0)
##  bindrcpp     * 0.2.2    2018-03-29 [1] CRAN (R 3.5.0)
##  broom          0.5.1    2018-12-05 [1] CRAN (R 3.5.1)
##  callr          3.0.0    2018-08-24 [1] CRAN (R 3.5.0)
##  cli            1.0.1    2018-09-25 [1] CRAN (R 3.5.0)
##  colorspace     1.3-2    2016-12-14 [1] CRAN (R 3.5.0)
##  crayon         1.3.4    2017-09-16 [1] CRAN (R 3.5.0)
##  curl           3.2      2018-03-28 [1] CRAN (R 3.5.0)
##  desc           1.2.0    2018-05-01 [1] CRAN (R 3.5.0)
##  devtools       2.0.1    2018-10-26 [1] CRAN (R 3.5.1)
##  digest         0.6.18   2018-10-10 [1] CRAN (R 3.5.0)
##  dplyr        * 0.7.8    2018-11-10 [1] CRAN (R 3.5.0)
##  evaluate       0.12     2018-10-09 [1] CRAN (R 3.5.0)
##  fansi          0.4.0    2018-10-05 [1] CRAN (R 3.5.0)
##  farver         1.1.0    2018-11-20 [1] CRAN (R 3.5.0)
##  forcats      * 0.3.0    2018-02-19 [1] CRAN (R 3.5.0)
##  fs             1.2.6    2018-08-23 [1] CRAN (R 3.5.0)
##  generics       0.0.2    2018-11-29 [1] CRAN (R 3.5.0)
##  ggforce        0.1.3    2018-07-07 [1] CRAN (R 3.5.0)
##  ggplot2      * 3.1.0    2018-10-25 [1] CRAN (R 3.5.0)
##  ggraph       * 1.0.2    2018-07-07 [1] CRAN (R 3.5.0)
##  ggrepel      * 0.8.0    2018-05-09 [1] CRAN (R 3.5.0)
##  glue           1.3.0    2018-07-17 [1] CRAN (R 3.5.0)
##  gridExtra      2.3      2017-09-09 [1] CRAN (R 3.5.0)
##  gtable         0.2.0    2016-02-26 [1] CRAN (R 3.5.0)
##  here         * 0.1      2017-05-28 [1] CRAN (R 3.5.0)
##  highr          0.7      2018-06-09 [1] CRAN (R 3.5.0)
##  htmltools      0.3.6    2017-04-28 [1] CRAN (R 3.5.0)
##  httr           1.3.1    2017-08-20 [1] CRAN (R 3.5.0)
##  igraph       * 1.2.2    2018-07-27 [1] CRAN (R 3.5.0)
##  janeaustenr    0.1.5    2017-06-10 [1] CRAN (R 3.5.0)
##  jsonlite       1.5      2017-06-01 [1] CRAN (R 3.5.0)
##  knitr        * 1.20     2018-02-20 [1] CRAN (R 3.5.0)
##  labeling       0.3      2014-08-23 [1] CRAN (R 3.5.0)
##  lattice        0.20-38  2018-11-04 [1] CRAN (R 3.5.1)
##  lazyeval       0.2.1    2017-10-29 [1] CRAN (R 3.5.0)
##  lubridate    * 1.7.4    2018-04-11 [1] CRAN (R 3.5.0)
##  magrittr       1.5      2014-11-22 [1] CRAN (R 3.5.0)
##  MASS           7.3-51.1 2018-11-01 [1] CRAN (R 3.5.0)
##  Matrix         1.2-15   2018-11-01 [1] CRAN (R 3.5.0)
##  memoise        1.1.0    2017-04-21 [1] CRAN (R 3.5.0)
##  modeltools     0.2-22   2018-07-16 [1] CRAN (R 3.5.0)
##  munsell        0.5.0    2018-06-12 [1] CRAN (R 3.5.0)
##  nlme           3.1-137  2018-04-07 [1] CRAN (R 3.5.1)
##  NLP            0.2-0    2018-10-18 [1] CRAN (R 3.5.0)
##  openssl        1.1      2018-11-15 [1] CRAN (R 3.5.0)
##  pillar         1.3.0    2018-07-14 [1] CRAN (R 3.5.0)
##  pkgbuild       1.0.2    2018-10-16 [1] CRAN (R 3.5.0)
##  pkgconfig      2.0.2    2018-08-16 [1] CRAN (R 3.5.0)
##  pkgload        1.0.2    2018-10-29 [1] CRAN (R 3.5.0)
##  plyr           1.8.4    2016-06-08 [1] CRAN (R 3.5.0)
##  prettyunits    1.0.2    2015-07-13 [1] CRAN (R 3.5.0)
##  processx       3.2.1    2018-12-05 [1] CRAN (R 3.5.1)
##  ps             1.2.1    2018-11-06 [1] CRAN (R 3.5.0)
##  purrr        * 0.2.5    2018-05-29 [1] CRAN (R 3.5.0)
##  R6             2.3.0    2018-10-04 [1] CRAN (R 3.5.0)
##  RColorBrewer * 1.1-2    2014-12-07 [1] CRAN (R 3.5.0)
##  Rcpp           1.0.0    2018-11-07 [1] CRAN (R 3.5.0)
##  remotes        2.0.2    2018-10-30 [1] CRAN (R 3.5.0)
##  reshape2       1.4.3    2017-12-11 [1] CRAN (R 3.5.0)
##  rlang          0.3.0.1  2018-10-25 [1] CRAN (R 3.5.0)
##  rmarkdown      1.10     2018-06-11 [1] CRAN (R 3.5.0)
##  rprojroot      1.3-2    2018-01-03 [1] CRAN (R 3.5.0)
##  rtweet       * 0.6.8    2018-09-28 [1] CRAN (R 3.5.0)
##  scales         1.0.0    2018-08-09 [1] CRAN (R 3.5.0)
##  sessioninfo    1.1.1    2018-11-05 [1] CRAN (R 3.5.0)
##  slam           0.1-43   2018-04-23 [1] CRAN (R 3.5.0)
##  SnowballC      0.5.1    2014-08-09 [1] CRAN (R 3.5.0)
##  stringi        1.2.4    2018-07-20 [1] CRAN (R 3.5.0)
##  stringr      * 1.3.1    2018-05-10 [1] CRAN (R 3.5.0)
##  testthat       2.0.1    2018-10-13 [1] CRAN (R 3.5.0)
##  tibble         1.4.2    2018-01-22 [1] CRAN (R 3.5.0)
##  tidyr        * 0.8.2    2018-10-28 [1] CRAN (R 3.5.0)
##  tidyselect     0.2.5    2018-10-11 [1] CRAN (R 3.5.0)
##  tidytext     * 0.2.0    2018-10-17 [1] CRAN (R 3.5.0)
##  tm             0.7-5    2018-07-29 [1] CRAN (R 3.5.0)
##  tokenizers     0.2.1    2018-03-29 [1] CRAN (R 3.5.0)
##  topicmodels  * 0.2-7    2017-11-03 [1] CRAN (R 3.5.0)
##  tweenr         1.0.0    2018-09-27 [1] CRAN (R 3.5.0)
##  units          0.6-2    2018-12-05 [1] CRAN (R 3.5.0)
##  usethis        1.4.0    2018-08-14 [1] CRAN (R 3.5.0)
##  utf8           1.1.4    2018-05-24 [1] CRAN (R 3.5.0)
##  viridis      * 0.5.1    2018-03-29 [1] CRAN (R 3.5.0)
##  viridisLite  * 0.3.0    2018-02-01 [1] CRAN (R 3.5.0)
##  withr          2.1.2    2018-03-15 [1] CRAN (R 3.5.0)
##  wordcloud    * 2.6      2018-08-24 [1] CRAN (R 3.5.0)
##  xml2           1.2.0    2018-01-24 [1] CRAN (R 3.5.0)
##  yaml           2.2.0    2018-07-25 [1] CRAN (R 3.5.0)
## 
## [1] /Library/Frameworks/R.framework/Versions/3.5/Resources/library</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
